{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Installed kernelspec python3 in /gpfs/data01/glasslab/home/jhauk/.local/share/jupyter/kernels/python3\r\n"
     ]
    }
   ],
   "source": [
    "import os\n",
    "os.environ[\"PATH\"] += \"/bioinformatics/anaconda3_052020/condabin:/bioinformatics/A.C.Rsuite:/bioinformatics/bowtie2:/bioinformatics/BSseeker2:/bioinformatics/adapterremoval:/bioinformatics/idr/bin:/bioinformatics/glassutils/scripts:/bioinformatics/STAR/bin/Linux_x86_64_static:/bioinformatics/scripts:/bioinformatics/sratoolkit/bin:/bioinformatics/FastQC:/bioinformatics/bedtools/bin:/bioinformatics/samtools/bin:/bioinformatics/anaconda3_052020/bin:/bioinformatics/homer/bin:/usr/local/sbin:/usr/sbin:/usr/bin:/usr/local/bin:/usr/local/lib:/bioinformatics/anaconda3_052020/condabin:/bioinformatics/A.C.Rsuite:/bioinformatics/bowtie2:/bioinformatics/BSseeker2:/bioinformatics/adapterremoval:/bioinformatics/idr/bin:/bioinformatics/glassutils/scripts:/bioinformatics/STAR/bin/Linux_x86_64_static:/bioinformatics/scripts:/bioinformatics/sratoolkit/bin:/bioinformatics/FastQC:/bioinformatics/bedtools/bin:/bioinformatics/samtools/bin:/bioinformatics/anaconda3_052020/bin:/bioinformatics/homer/bin:/usr/local/sbin:/usr/sbin:/usr/bin:/usr/local/bin:/usr/local/lib:\"\n",
    "!python -m ipykernel install --user"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\tPeak file = /data/home/nspann/NJS_epiglass_scratch/NJS201116v2_BMDM_WT_LXRPh198mut_veh_GW3965_PU1_p300_h3k4me2_PhS198LXR_h3k27Ac/tag_directories/BMDM_WT_ChIP_H3K27Ac_p300_LXR_NJS201116/BMDM_ATAC_mergedTotal_95029_peaks.txt\n",
      "\tGenome = mm10\n",
      "\tOrganism = mouse\n",
      "\tTag Directories:\n",
      "\t\t/gpfs/data01/glasslab/home/jhauk/LXR_code/10_31_24_Our_Data/tag_directories/WT_BMDM_ChIP_H3K27ac_GW_1uM_rep3_SY_l20241028_AAGTTGGA_AATACAGG\n",
      "\t\t/gpfs/data01/glasslab/home/jhauk/LXR_code/10_31_24_Our_Data/tag_directories/WT_BMDM_ChIP_H3K27ac_vehicle_DMSO_rep3_SY_l20241028_AAGTAAGT_AATAGTCC\n",
      "\tPeak Region set to 1000\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 0\n",
      "\t\tpeakfile formatted lines: 95029\n",
      "\t\tDuplicated Peak IDs: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 95029\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tResizing peaks...\n",
      "\tReading Positions...\n",
      "\t-----------------------\n",
      "\tFinding Closest TSS...\n",
      "\tAnnotating:........................\n",
      "\t\tAnnotation\tNumber of peaks\tTotal size (bp)\tLog2 Ratio (obs/exp)\tLogP enrichment (+values depleted)\n",
      "\t\t3UTR\t1000.0\t20049228\t0.515\t-60.336\n",
      "\t\tmiRNA\t5.0\t24864\t2.526\t-6.210\n",
      "\t\tncRNA\t265.0\t3372538\t1.170\t-70.954\n",
      "\t\tTTS\t1440.0\t27011103\t0.611\t-117.384\n",
      "\t\tpseudo\t20.0\t528823\t0.115\t-0.940\n",
      "\t\tExon\t1646.0\t34014877\t0.471\t-83.360\n",
      "\t\tIntron\t41037.0\t942718221\t0.318\t-1489.645\n",
      "\t\tIntergenic\t33923.0\t1662058715\t-0.774\t12449.631\n",
      "\t\tPromoter\t14837.0\t29761709\t3.836\t-26722.391\n",
      "\t\t5UTR\t854.0\t2270504\t3.429\t-1262.640\n",
      "\t\tsnoRNA\t0.0\t19\t-0.001\t0.001\n",
      "\t\trRNA\t1.0\t5631\t2.347\t-1.723\n",
      "\tNOTE: If this part takes more than 2 minutes, there is a good chance\n",
      "\t\tyour machine ran out of memory: consider hitting ctrl+C and rerunning\n",
      "\t\tthe command with \"-noann\"\n",
      "\tTo capture annotation stats in a file, use \"-annStats <filename>\" next time\n",
      "\tAnnotating:.........................\n",
      "\t\tAnnotation\tNumber of peaks\tTotal size (bp)\tLog2 Ratio (obs/exp)\tLogP enrichment (+values depleted)\n",
      "\t\t3UTR\t1000.0\t20049228\t0.515\t-60.478\n",
      "\t\tOther\t145.0\t7955052\t-0.937\t41.205\n",
      "\t\tRC?\t1.0\t10979\t1.384\t-1.145\n",
      "\t\tRNA\t3.0\t114021\t-0.407\t0.827\n",
      "\t\tmiRNA\t5.0\t24864\t2.527\t-6.212\n",
      "\t\tncRNA\t264.0\t3372538\t1.166\t-70.206\n",
      "\t\tTTS\t1440.0\t27011103\t0.611\t-117.617\n",
      "\t\tLINE\t3759.0\t543770301\t-2.336\t10565.922\n",
      "\t\tLINE?\t0.0\t8168\t-0.362\t0.285\n",
      "\t\tsrpRNA\t1.0\t43307\t-0.596\t0.590\n",
      "\t\tSINE\t3816.0\t196566416\t-0.846\t861.925\n",
      "\t\tRC\t1.0\t65629\t-1.196\t1.099\n",
      "\t\ttRNA\t34.0\t266576\t1.870\t-21.751\n",
      "\t\tDNA?\t1.0\t142206\t-2.311\t3.177\n",
      "\t\tpseudo\t20.0\t528823\t0.116\t-0.942\n",
      "\t\tDNA\t685.0\t28691693\t-0.548\t59.810\n",
      "\t\tExon\t1645.0\t34014877\t0.471\t-83.243\n",
      "\t\tIntron\t31511.0\t598538808\t0.593\t-3138.823\n",
      "\t\tIntergenic\t24886.0\t828464587\t-0.216\t418.465\n",
      "\t\tPromoter\t14834.0\t29761709\t3.836\t-26720.389\n",
      "\t\t5UTR\t853.0\t2270504\t3.428\t-1260.614\n",
      "\t\tsnoRNA\t0.0\t19\t-0.001\t0.001\n",
      "\t\tLTR?\t5.0\t192563\t-0.427\t1.085\n",
      "\t\tscRNA\t8.0\t602998\t-1.395\t6.836\n",
      "\t\tCpG-Island\t2275.0\t3193893\t4.351\t-4727.624\n",
      "\t\tLow_complexity\t611.0\t19402818\t-0.148\t5.270\n",
      "\t\tLTR\t6789.0\t313071081\t-0.686\t1010.741\n",
      "\t\tSimple_repeat\t329.0\t57557030\t-2.610\t1103.094\n",
      "\t\tsnRNA\t12.0\t237524\t0.534\t-2.011\n",
      "\t\tUnknown\t19.0\t2407459\t-2.145\t38.935\n",
      "\t\tSINE?\t0.0\t29758\t-1.027\t1.038\n",
      "\t\tSatellite\t52.0\t4573395\t-1.618\t51.853\n",
      "\t\trRNA\t25.0\t166488\t2.105\t-19.576\n",
      "\tCounting Tags in Peaks from each directory...\n",
      "\tRatio for /gpfs/data01/glasslab/home/jhauk/LXR_code/10_31_24_Our_Data/tag_directories/WT_BMDM_ChIP_H3K27ac_GW_1uM_rep3_SY_l20241028_AAGTTGGA_AATACAGG : 1.06064885618036\n",
      "\tRatio for /gpfs/data01/glasslab/home/jhauk/LXR_code/10_31_24_Our_Data/tag_directories/WT_BMDM_ChIP_H3K27ac_vehicle_DMSO_rep3_SY_l20241028_AAGTAAGT_AATAGTCC : 0.341932642962209\n",
      "\tOrganism: mouse\n",
      "\tLoading Gene Informaiton...\n",
      "\tOutputing Annotation File...\n",
      "\tDone annotating peaks file\n",
      "\n"
     ]
    }
   ],
   "source": [
    "!annotatePeaks.pl /data/home/nspann/NJS_epiglass_scratch/NJS201116v2_BMDM_WT_LXRPh198mut_veh_GW3965_PU1_p300_h3k4me2_PhS198LXR_h3k27Ac/tag_directories/BMDM_WT_ChIP_H3K27Ac_p300_LXR_NJS201116/BMDM_ATAC_mergedTotal_95029_peaks.txt mm10 -d /gpfs/data01/glasslab/home/jhauk/LXR_code/10_31_24_Our_Data/tag_directories/WT_BMDM_ChIP_H3K27ac_GW_1uM_rep3_SY_l20241028_AAGTTGGA_AATACAGG /gpfs/data01/glasslab/home/jhauk/LXR_code/10_31_24_Our_Data/tag_directories/WT_BMDM_ChIP_H3K27ac_vehicle_DMSO_rep3_SY_l20241028_AAGTAAGT_AATAGTCC -size 1000 > normalized_counts_Nathan_BMDM_K27ac_GW_gained"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Output file: normalized_counts_Nathan_BMDM_K27ac_GW_gained_filtered_20.txt\r\n"
     ]
    }
   ],
   "source": [
    "!filter_low_tag_counts.pl normalized_counts_Nathan_BMDM_K27ac_GW_gained 2 20 one"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\tPeak file = normalized_counts_Nathan_BMDM_K27ac_GW_gained_filtered_20.txt\n",
      "\tGenome = mm10\n",
      "\tOrganism = mouse\n",
      "\tTag Directories:\n",
      "\t\t/gpfs/data01/glasslab/home/jhauk/LXR_code/10_31_24_Our_Data/tag_directories/WT_BMDM_ChIP_H3K27ac_GW_1uM_rep3_SY_l20241028_AAGTTGGA_AATACAGG\n",
      "\t\t/gpfs/data01/glasslab/home/jhauk/LXR_code/10_31_24_Our_Data/tag_directories/WT_BMDM_ChIP_H3K27ac_vehicle_DMSO_rep3_SY_l20241028_AAGTAAGT_AATAGTCC\n",
      "\tPeak Region set to 1000\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 0\n",
      "\t\tpeakfile formatted lines: 50336\n",
      "\t\tDuplicated Peak IDs: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 50336\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tResizing peaks...\n",
      "\tReading Positions...\n",
      "\t-----------------------\n",
      "\tFinding Closest TSS...\n",
      "\tAnnotating:........................\n",
      "\t\tAnnotation\tNumber of peaks\tTotal size (bp)\tLog2 Ratio (obs/exp)\tLogP enrichment (+values depleted)\n",
      "\t\t3UTR\t456.0\t20049228\t0.298\t-11.542\n",
      "\t\tmiRNA\t1.0\t24864\t1.121\t-0.998\n",
      "\t\tncRNA\t152.0\t3372538\t1.285\t-48.761\n",
      "\t\tTTS\t686.0\t27011103\t0.458\t-34.384\n",
      "\t\tpseudo\t6.0\t528823\t-0.705\t1.934\n",
      "\t\tExon\t820.0\t34014877\t0.382\t-29.629\n",
      "\t\tIntron\t20961.0\t942718221\t0.266\t-535.274\n",
      "\t\tIntergenic\t14087.0\t1662058715\t-1.126\t11305.548\n",
      "\t\tPromoter\t12521.0\t29761709\t4.508\t-28727.679\n",
      "\t\t5UTR\t645.0\t2270504\t3.941\t-1166.729\n",
      "\t\tsnoRNA\t0.0\t19\t-0.001\t0.000\n",
      "\t\trRNA\t1.0\t5631\t3.263\t-2.314\n",
      "\tNOTE: If this part takes more than 2 minutes, there is a good chance\n",
      "\t\tyour machine ran out of memory: consider hitting ctrl+C and rerunning\n",
      "\t\tthe command with \"-noann\"\n",
      "\tTo capture annotation stats in a file, use \"-annStats <filename>\" next time\n",
      "\tAnnotating:........................\n",
      "\t\tAnnotation\tNumber of peaks\tTotal size (bp)\tLog2 Ratio (obs/exp)\tLogP enrichment (+values depleted)\n",
      "\t\t3UTR\t456.0\t20049228\t0.299\t-11.584\n",
      "\t\tOther\t43.0\t7955052\t-1.774\t53.745\n",
      "\t\tRC?\t1.0\t10979\t2.301\t-1.695\n",
      "\t\tRNA\t2.0\t114021\t-0.076\t0.435\n",
      "\t\tmiRNA\t1.0\t24864\t1.122\t-0.998\n",
      "\t\tncRNA\t151.0\t3372538\t1.276\t-47.906\n",
      "\t\tTTS\t686.0\t27011103\t0.458\t-34.473\n",
      "\t\tLINE\t1717.0\t543765706\t-2.549\t6112.944\n",
      "\t\tLINE?\t0.0\t8168\t-0.203\t0.151\n",
      "\t\tsrpRNA\t1.0\t43307\t0.321\t-0.596\n",
      "\t\tSINE\t1331.0\t196566090\t-1.449\t1025.750\n",
      "\t\tRC\t1.0\t65629\t-0.279\t0.419\n",
      "\t\ttRNA\t28.0\t266576\t2.506\t-27.982\n",
      "\t\tDNA?\t1.0\t142206\t-1.394\t1.340\n",
      "\t\tpseudo\t6.0\t528823\t-0.704\t1.931\n",
      "\t\tDNA\t288.0\t28691693\t-0.881\t70.074\n",
      "\t\tExon\t819.0\t34014877\t0.381\t-29.448\n",
      "\t\tIntron\t16151.0\t598538808\t0.546\t-1372.073\n",
      "\t\tIntergenic\t10294.0\t828448942\t-0.573\t1279.336\n",
      "\t\tPromoter\t12518.0\t29761709\t4.508\t-28723.113\n",
      "\t\t5UTR\t644.0\t2270504\t3.940\t-1164.265\n",
      "\t\tsnoRNA\t0.0\t19\t-0.001\t0.000\n",
      "\t\tLTR?\t4.0\t192563\t0.168\t-0.742\n",
      "\t\tscRNA\t4.0\t602998\t-1.479\t4.292\n",
      "\t\tCpG-Island\t1765.0\t3193893\t4.902\t-4324.879\n",
      "\t\tLow_complexity\t293.0\t19402657\t-0.292\t8.579\n",
      "\t\tLTR\t2890.0\t313068869\t-1.002\t986.797\n",
      "\t\tSimple_repeat\t172.0\t57556995\t-2.629\t589.856\n",
      "\t\tsnRNA\t7.0\t237524\t0.673\t-1.863\n",
      "\t\tUnknown\t14.0\t2407459\t-1.668\t16.201\n",
      "\t\tSINE?\t0.0\t29758\t-0.632\t0.550\n",
      "\t\tSatellite\t27.0\t4573395\t-1.647\t28.948\n",
      "\t\trRNA\t21.0\t166488\t2.771\t-24.705\n",
      "\tCounting Tags in Peaks from each directory...\n",
      "\tRatio for /gpfs/data01/glasslab/home/jhauk/LXR_code/10_31_24_Our_Data/tag_directories/WT_BMDM_ChIP_H3K27ac_GW_1uM_rep3_SY_l20241028_AAGTTGGA_AATACAGG : 1.06064885618036\n",
      "\tRatio for /gpfs/data01/glasslab/home/jhauk/LXR_code/10_31_24_Our_Data/tag_directories/WT_BMDM_ChIP_H3K27ac_vehicle_DMSO_rep3_SY_l20241028_AAGTAAGT_AATAGTCC : 0.341932642962209\n",
      "\tOrganism: mouse\n",
      "\tLoading Gene Informaiton...\n",
      "\tOutputing Annotation File...\n",
      "\tDone annotating peaks file\n",
      "\n"
     ]
    }
   ],
   "source": [
    "!annotatePeaks.pl normalized_counts_Nathan_BMDM_K27ac_GW_gained_filtered_20.txt mm10 -d /gpfs/data01/glasslab/home/jhauk/LXR_code/10_31_24_Our_Data/tag_directories/WT_BMDM_ChIP_H3K27ac_GW_1uM_rep3_SY_l20241028_AAGTTGGA_AATACAGG /gpfs/data01/glasslab/home/jhauk/LXR_code/10_31_24_Our_Data/tag_directories/WT_BMDM_ChIP_H3K27ac_vehicle_DMSO_rep3_SY_l20241028_AAGTAAGT_AATAGTCC -size 1000 > normalized_counts_Nathan_BMDM_K27ac_GW_gained_filtered_20_annotated"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\tDifferential Peaks: 3668 of 50336 (7.29% passed)\r\n"
     ]
    }
   ],
   "source": [
    "!getDifferentialPeaks normalized_counts_Nathan_BMDM_K27ac_GW_gained_filtered_20_annotated /gpfs/data01/glasslab/home/jhauk/LXR_code/10_31_24_Our_Data/tag_directories/WT_BMDM_ChIP_H3K27ac_GW_1uM_rep3_SY_l20241028_AAGTTGGA_AATACAGG /gpfs/data01/glasslab/home/jhauk/LXR_code/10_31_24_Our_Data/tag_directories/WT_BMDM_ChIP_H3K27ac_vehicle_DMSO_rep3_SY_l20241028_AAGTAAGT_AATAGTCC -F 1.5 -size 1000 -P 0.0001 > getDiff_Nathan_BMDM_K27ac_GW_gained_20_FC1_5_p0001.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\tPosition file = getDiff_Nathan_BMDM_K27ac_GW_gained_20_FC1_5_p0001.txt\n",
      "\tGenome = mm10\n",
      "\tOutput Directory = getDiff_Nathan_BMDM_K27ac_GW_gained_20_FC1_5_p0001_GC_matched.txt\n",
      "\tFragment size set to 200\n",
      "\tMotif length set at 8,10,12,14,*** might want to hit ctrl+C to quit if it takes too long!\n",
      "*** If you run out of memmory try reducing the number background sequences\n",
      "16,*** might want to hit ctrl+C to quit if it takes too long!\n",
      "*** If you run out of memmory try reducing the number background sequences\n",
      "\n",
      "\tWill search for strings with 3 mismatches\n",
      "\tUsing 24 CPUs\n",
      "\tFound mset for \"mouse\", will check against vertebrates motifs\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 0\n",
      "\t\tpeakfile formatted lines: 3668\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 3668\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground files for 200 bp fragments found.\n",
      "\tCustom genome sequence directory: /bioinformatics/homer_4.10//data/genomes/mm10//\n",
      "\n",
      "\tExtracting sequences from directory: /bioinformatics/homer_4.10//data/genomes/mm10//\n",
      "\tExtracting 185 sequences from chr1\n",
      "\tExtracting 274 sequences from chr2\n",
      "\tExtracting 153 sequences from chr3\n",
      "\tExtracting 277 sequences from chr4\n",
      "\tExtracting 238 sequences from chr5\n",
      "\tExtracting 166 sequences from chr6\n",
      "\tExtracting 262 sequences from chr7\n",
      "\tExtracting 211 sequences from chr8\n",
      "\tExtracting 241 sequences from chr9\n",
      "\tExtracting 171 sequences from chr10\n",
      "\tExtracting 317 sequences from chr11\n",
      "\tExtracting 149 sequences from chr12\n",
      "\tExtracting 157 sequences from chr13\n",
      "\tExtracting 128 sequences from chr14\n",
      "\tExtracting 157 sequences from chr15\n",
      "\tExtracting 93 sequences from chr16\n",
      "\tExtracting 199 sequences from chr17\n",
      "\tExtracting 97 sequences from chr18\n",
      "\tExtracting 123 sequences from chr19\n",
      "\tExtracting 68 sequences from chrX\n",
      "\tExtracting 2 sequences from chr4_GL456216_random\n",
      "\t\t158 of 3668 removed because they had >70.00% Ns (i.e. masked repeats)\n",
      "\n",
      "\tNot removing redundant sequences\n",
      "\n",
      "\n",
      "\tSequences processed:\n",
      "\t\tAuto detected maximum sequence length of 201 bp\n",
      "\t\t3510 total\n",
      "\n",
      "\tFrequency Bins: 0.2 0.25 0.3 0.35 0.4 0.45 0.5 0.6 0.7 0.8\n",
      "\tFreq\tBin\tCount\n",
      "\t0.3\t2\t1\n",
      "\t0.35\t3\t13\n",
      "\t0.4\t4\t45\n",
      "\t0.45\t5\t116\n",
      "\t0.5\t6\t257\n",
      "\t0.6\t7\t967\n",
      "\t0.7\t8\t1320\n",
      "\t0.8\t9\t770\n",
      "\t10\t10\t21\n",
      "\n",
      "\tTotal sequences set to 50000\n",
      "\n",
      "\tChoosing background that matches in CpG/GC content...\n",
      "\tBin\t# Targets\t# Background\tBackground Weight\n",
      "\t2\t1\t13\t1.019\n",
      "\t3\t13\t172\t1.001\n",
      "\t4\t45\t596\t1.000\n",
      "\t5\t116\t1536\t1.000\n",
      "\t6\t257\t3404\t1.000\n",
      "\t7\t967\t12808\t1.000\n",
      "\t8\t1320\t17483\t1.000\n",
      "\t9\t770\t10199\t1.000\n",
      "\t10\t21\t278\t1.001\n",
      "\tAssembling sequence file...\n",
      "\tNormalizing lower order oligos using homer2\n",
      "\n",
      "\tReading input files...\n",
      "\t32625 total sequences read\n",
      "\tAutonormalization: 1-mers (4 total)\n",
      "\t\tA\t19.03%\t17.81%\t1.068\n",
      "\t\tC\t30.97%\t32.19%\t0.962\n",
      "\t\tG\t30.97%\t32.19%\t0.962\n",
      "\t\tT\t19.03%\t17.81%\t1.068\n",
      "\tAutonormalization: 2-mers (16 total)\n",
      "\t\tAA\t4.28%\t3.54%\t1.210\n",
      "\t\tCA\t6.20%\t6.30%\t0.984\n",
      "\t\tGA\t6.12%\t5.95%\t1.028\n",
      "\t\tTA\t2.40%\t2.02%\t1.186\n",
      "\t\tAC\t4.65%\t4.54%\t1.023\n",
      "\t\tCC\t10.20%\t11.14%\t0.916\n",
      "\t\tGC\t10.02%\t10.54%\t0.951\n",
      "\t\tTC\t6.12%\t5.95%\t1.028\n",
      "\t\tAG\t7.42%\t7.36%\t1.008\n",
      "\t\tCG\t7.19%\t7.38%\t0.975\n",
      "\t\tGG\t10.20%\t11.14%\t0.916\n",
      "\t\tTG\t6.20%\t6.30%\t0.984\n",
      "\t\tAT\t2.65%\t2.39%\t1.110\n",
      "\t\tCT\t7.42%\t7.36%\t1.008\n",
      "\t\tGT\t4.65%\t4.54%\t1.023\n",
      "\t\tTT\t4.28%\t3.54%\t1.210\n",
      "\tAutonormalization: 3-mers (64 total)\n",
      "\tNormalization weights can be found in file: getDiff_Nathan_BMDM_K27ac_GW_gained_20_FC1_5_p0001_GC_matched.txt/seq.autonorm.tsv\n",
      "\tConverging on autonormalization solution:\n",
      "\t...............................................................................\n",
      "\tFinal normalization:\tAutonormalization: 1-mers (4 total)\n",
      "\t\tA\t19.03%\t18.99%\t1.002\n",
      "\t\tC\t30.97%\t31.01%\t0.999\n",
      "\t\tG\t30.97%\t31.01%\t0.999\n",
      "\t\tT\t19.03%\t18.99%\t1.002\n",
      "\tAutonormalization: 2-mers (16 total)\n",
      "\t\tAA\t4.28%\t4.24%\t1.009\n",
      "\t\tCA\t6.20%\t6.23%\t0.995\n",
      "\t\tGA\t6.12%\t6.10%\t1.003\n",
      "\t\tTA\t2.40%\t2.39%\t1.004\n",
      "\t\tAC\t4.65%\t4.65%\t1.000\n",
      "\t\tCC\t10.20%\t10.27%\t0.993\n",
      "\t\tGC\t10.02%\t10.01%\t1.001\n",
      "\t\tTC\t6.12%\t6.10%\t1.003\n",
      "\t\tAG\t7.42%\t7.40%\t1.002\n",
      "\t\tCG\t7.19%\t7.13%\t1.008\n",
      "\t\tGG\t10.20%\t10.27%\t0.993\n",
      "\t\tTG\t6.20%\t6.23%\t0.995\n",
      "\t\tAT\t2.65%\t2.68%\t0.990\n",
      "\t\tCT\t7.42%\t7.40%\t1.002\n",
      "\t\tGT\t4.65%\t4.65%\t1.000\n",
      "\t\tTT\t4.28%\t4.24%\t1.009\n",
      "\tAutonormalization: 3-mers (64 total)\n",
      "\tFinished preparing sequence/group files\n",
      "\n",
      "\t----------------------------------------------------------\n",
      "\tKnown motif enrichment\n",
      "\n",
      "\tReading input files...\n",
      "\t32625 total sequences read\n",
      "\t428 motifs loaded\n",
      "\tCache length = 11180\n",
      "\tUsing binomial scoring\n",
      "\tChecking enrichment of 428 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tPreparing HTML output with sequence logos...\n",
      "\t\t1 of 428 (1e-132) Elf4(ETS)/BMDM-Elf4-ChIP-Seq(GSE88699)/Homer\n",
      "\t\t2 of 428 (1e-118) ETS1(ETS)/Jurkat-ETS1-ChIP-Seq(GSE17954)/Homer\n",
      "\t\t3 of 428 (1e-117) Sp1(Zf)/Promoter/Homer\n",
      "\t\t4 of 428 (1e-106) Fli1(ETS)/CD8-FLI-ChIP-Seq(GSE20898)/Homer\n",
      "\t\t5 of 428 (1e-106) Sp2(Zf)/HEK293-Sp2.eGFP-ChIP-Seq(Encode)/Homer\n",
      "\t\t6 of 428 (1e-106) GABPA(ETS)/Jurkat-GABPa-ChIP-Seq(GSE17954)/Homer\n",
      "\t\t7 of 428 (1e-104) ELF5(ETS)/T47D-ELF5-ChIP-Seq(GSE30407)/Homer\n",
      "\t\t8 of 428 (1e-104) Sp5(Zf)/mES-Sp5.Flag-ChIP-Seq(GSE72989)/Homer\n",
      "\t\t9 of 428 (1e-104) ETV1(ETS)/GIST48-ETV1-ChIP-Seq(GSE22441)/Homer\n",
      "\t\t10 of 428 (1e-103) PU.1(ETS)/ThioMac-PU.1-ChIP-Seq(GSE21512)/Homer\n",
      "\t\t11 of 428 (1e-103) EHF(ETS)/LoVo-EHF-ChIP-Seq(GSE49402)/Homer\n",
      "\t\t12 of 428 (1e-100) ERG(ETS)/VCaP-ERG-ChIP-Seq(GSE14097)/Homer\n",
      "\t\t13 of 428 (1e-100) SpiB(ETS)/OCILY3-SPIB-ChIP-Seq(GSE56857)/Homer\n",
      "\t\t14 of 428 (1e-98) Etv2(ETS)/ES-ER71-ChIP-Seq(GSE59402)/Homer\n",
      "\t\t15 of 428 (1e-90) KLF3(Zf)/MEF-Klf3-ChIP-Seq(GSE44748)/Homer\n",
      "\t\t16 of 428 (1e-89) ETV4(ETS)/HepG2-ETV4-ChIP-Seq(ENCODE)/Homer\n",
      "\t\t17 of 428 (1e-81) KLF5(Zf)/LoVo-KLF5-ChIP-Seq(GSE49402)/Homer\n",
      "\t\t18 of 428 (1e-74) ELF1(ETS)/Jurkat-ELF1-ChIP-Seq(SRA014231)/Homer\n",
      "\t\t19 of 428 (1e-74) ETS(ETS)/Promoter/Homer\n",
      "\t\t20 of 428 (1e-69) NFY(CCAAT)/Promoter/Homer\n",
      "\t\t21 of 428 (1e-69) ELF3(ETS)/PDAC-ELF3-ChIP-Seq(GSE64557)/Homer\n",
      "\t\t22 of 428 (1e-67) KLF14(Zf)/HEK293-KLF14.GFP-ChIP-Seq(GSE58341)/Homer\n",
      "\t\t23 of 428 (1e-65) Elk4(ETS)/Hela-Elk4-ChIP-Seq(GSE31477)/Homer\n",
      "\t\t24 of 428 (1e-64) Elk1(ETS)/Hela-Elk1-ChIP-Seq(GSE31477)/Homer\n",
      "\t\t25 of 428 (1e-63) KLF6(Zf)/PDAC-KLF6-ChIP-Seq(GSE64557)/Homer\n",
      "\t\t26 of 428 (1e-50) Klf9(Zf)/GBM-Klf9-ChIP-Seq(GSE62211)/Homer\n",
      "\t\t27 of 428 (1e-47) CTCF(Zf)/CD4+-CTCF-ChIP-Seq(Barski_et_al.)/Homer\n",
      "\t\t28 of 428 (1e-45) Ronin(THAP)/ES-Thap11-ChIP-Seq(GSE51522)/Homer\n",
      "\t\t29 of 428 (1e-41) EWS:ERG-fusion(ETS)/CADO_ES1-EWS:ERG-ChIP-Seq(SRA014231)/Homer\n",
      "\t\t30 of 428 (1e-41) GFY-Staf(?,Zf)/Promoter/Homer\n",
      "\t\t31 of 428 (1e-41) Ets1-distal(ETS)/CD4+-PolII-ChIP-Seq(Barski_et_al.)/Homer\n",
      "\t\t32 of 428 (1e-41) EWS:FLI1-fusion(ETS)/SK_N_MC-EWS:FLI1-ChIP-Seq(SRA014231)/Homer\n",
      "\t\t33 of 428 (1e-41) PU.1-IRF(ETS:IRF)/Bcell-PU.1-ChIP-Seq(GSE21512)/Homer\n",
      "\t\t34 of 428 (1e-40) Klf4(Zf)/mES-Klf4-ChIP-Seq(GSE11431)/Homer\n",
      "\t\t35 of 428 (1e-36) LXRE(NR),DR4/RAW-LXRb.biotin-ChIP-Seq(GSE21512)/Homer\n",
      "\t\t36 of 428 (1e-36) SPDEF(ETS)/VCaP-SPDEF-ChIP-Seq(SRA014231)/Homer\n",
      "\t\t37 of 428 (1e-35) KLF10(Zf)/HEK293-KLF10.GFP-ChIP-Seq(GSE58341)/Homer\n",
      "\t\t38 of 428 (1e-34) PU.1:IRF8(ETS:IRF)/pDC-Irf8-ChIP-Seq(GSE66899)/Homer\n",
      "\t\t39 of 428 (1e-33) Atf1(bZIP)/K562-ATF1-ChIP-Seq(GSE31477)/Homer\n",
      "\t\t40 of 428 (1e-31) GFY(?)/Promoter/Homer\n",
      "\t\t41 of 428 (1e-28) Atf7(bZIP)/3T3L1-Atf7-ChIP-Seq(GSE56872)/Homer\n",
      "\t\t42 of 428 (1e-28) BORIS(Zf)/K562-CTCFL-ChIP-Seq(GSE32465)/Homer\n",
      "\t\t43 of 428 (1e-27) IRF8(IRF)/BMDM-IRF8-ChIP-Seq(GSE77884)/Homer\n",
      "\t\t44 of 428 (1e-26) NRF1(NRF)/MCF7-NRF1-ChIP-Seq(Unpublished)/Homer\n",
      "\t\t45 of 428 (1e-26) CRE(bZIP)/Promoter/Homer\n",
      "\t\t46 of 428 (1e-23) Atf2(bZIP)/3T3L1-Atf2-ChIP-Seq(GSE56872)/Homer\n",
      "\t\t47 of 428 (1e-21) Jun-AP1(bZIP)/K562-cJun-ChIP-Seq(GSE31477)/Homer\n",
      "\t\t48 of 428 (1e-21) NRF(NRF)/Promoter/Homer\n",
      "\t\t49 of 428 (1e-21) NFIL3(bZIP)/HepG2-NFIL3-ChIP-Seq(Encode)/Homer\n",
      "\t\t50 of 428 (1e-20) AP-1(bZIP)/ThioMac-PU.1-ChIP-Seq(GSE21512)/Homer\n",
      "\t\t51 of 428 (1e-20) ETS:RUNX(ETS,Runt)/Jurkat-RUNX1-ChIP-Seq(GSE17954)/Homer\n",
      "\t\t52 of 428 (1e-20) Fra1(bZIP)/BT549-Fra1-ChIP-Seq(GSE46166)/Homer\n",
      "\t\t53 of 428 (1e-20) Fra2(bZIP)/Striatum-Fra2-ChIP-Seq(GSE43429)/Homer\n",
      "\t\t54 of 428 (1e-20) JunB(bZIP)/DendriticCells-Junb-ChIP-Seq(GSE36099)/Homer\n",
      "\t\t55 of 428 (1e-20) c-Jun-CRE(bZIP)/K562-cJun-ChIP-Seq(GSE31477)/Homer\n",
      "\t\t56 of 428 (1e-19) BATF(bZIP)/Th17-BATF-ChIP-Seq(GSE39756)/Homer\n",
      "\t\t57 of 428 (1e-19) Fosl2(bZIP)/3T3L1-Fosl2-ChIP-Seq(GSE56872)/Homer\n",
      "\t\t58 of 428 (1e-19) JunD(bZIP)/K562-JunD-ChIP-Seq/Homer\n",
      "\t\t59 of 428 (1e-18) Atf3(bZIP)/GBM-ATF3-ChIP-Seq(GSE33912)/Homer\n",
      "\t\t60 of 428 (1e-17) YY1(Zf)/Promoter/Homer\n",
      "\t\t61 of 428 (1e-16) NFE2L2(bZIP)/HepG2-NFE2L2-ChIP-Seq(Encode)/Homer\n",
      "\t\t62 of 428 (1e-16) Nrf2(bZIP)/Lymphoblast-Nrf2-ChIP-Seq(GSE37589)/Homer\n",
      "\t\t63 of 428 (1e-15) Maz(Zf)/HepG2-Maz-ChIP-Seq(GSE31477)/Homer\n",
      "\t\t64 of 428 (1e-15) Bach1(bZIP)/K562-Bach1-ChIP-Seq(GSE31477)/Homer\n",
      "\t\t65 of 428 (1e-14) Bach2(bZIP)/OCILy7-Bach2-ChIP-Seq(GSE44420)/Homer\n",
      "\t\t66 of 428 (1e-14) Usf2(bHLH)/C2C12-Usf2-ChIP-Seq(GSE36030)/Homer\n",
      "\t\t67 of 428 (1e-13) IRF3(IRF)/BMDM-Irf3-ChIP-Seq(GSE67343)/Homer\n",
      "\t\t68 of 428 (1e-13) ZNF143|STAF(Zf)/CUTLL-ZNF143-ChIP-Seq(GSE29600)/Homer\n",
      "\t\t69 of 428 (1e-13) NF-E2(bZIP)/K562-NFE2-ChIP-Seq(GSE31477)/Homer\n",
      "\t\t70 of 428 (1e-11) Pknox1(Homeobox)/ES-Prep1-ChIP-Seq(GSE63282)/Homer\n",
      "\t\t71 of 428 (1e-10) MITF(bHLH)/MastCells-MITF-ChIP-Seq(GSE48085)/Homer\n",
      "\t\t72 of 428 (1e-10) Stat3+il21(Stat)/CD4-Stat3-ChIP-Seq(GSE19198)/Homer\n",
      "\t\t73 of 428 (1e-10) TFE3(bHLH)/MEF-TFE3-ChIP-Seq(GSE75757)/Homer\n",
      "\t\t74 of 428 (1e-9) EKLF(Zf)/Erythrocyte-Klf1-ChIP-Seq(GSE20478)/Homer\n",
      "\t\t75 of 428 (1e-9) MafK(bZIP)/C2C12-MafK-ChIP-Seq(GSE36030)/Homer\n",
      "\t\t76 of 428 (1e-9) Hoxa9(Homeobox)/ChickenMSG-Hoxa9.Flag-ChIP-Seq(GSE86088)/Homer\n",
      "\t\t77 of 428 (1e-9) Pbx3(Homeobox)/GM12878-PBX3-ChIP-Seq(GSE32465)/Homer\n",
      "\t\t78 of 428 (1e-9) MafA(bZIP)/Islet-MafA-ChIP-Seq(GSE30298)/Homer\n",
      "\t\t79 of 428 (1e-8) MafB(bZIP)/BMM-Mafb-ChIP-Seq(GSE75722)/Homer\n",
      "\t\t80 of 428 (1e-8) CEBP(bZIP)/ThioMac-CEBPb-ChIP-Seq(GSE21512)/Homer\n",
      "\t\t81 of 428 (1e-8) GFX(?)/Promoter/Homer\n",
      "\t\t82 of 428 (1e-8) USF1(bHLH)/GM12878-Usf1-ChIP-Seq(GSE32465)/Homer\n",
      "\t\t83 of 428 (1e-7) PBX2(Homeobox)/K562-PBX2-ChIP-Seq(Encode)/Homer\n",
      "\t\t84 of 428 (1e-7) Nanog(Homeobox)/mES-Nanog-ChIP-Seq(GSE11724)/Homer\n",
      "\t\t85 of 428 (1e-7) PBX1(Homeobox)/MCF7-PBX1-ChIP-Seq(GSE28007)/Homer\n",
      "\t\t86 of 428 (1e-6) IRF1(IRF)/PBMC-IRF1-ChIP-Seq(GSE43036)/Homer\n",
      "\t\t87 of 428 (1e-6) HLF(bZIP)/HSC-HLF.Flag-ChIP-Seq(GSE69817)/Homer\n",
      "\t\t88 of 428 (1e-6) IRF2(IRF)/Erythroblas-IRF2-ChIP-Seq(GSE36985)/Homer\n",
      "\t\t89 of 428 (1e-5) Gfi1b(Zf)/HPC7-Gfi1b-ChIP-Seq(GSE22178)/Homer\n",
      "\t\t90 of 428 (1e-5) CLOCK(bHLH)/Liver-Clock-ChIP-Seq(GSE39860)/Homer\n",
      "\t\t91 of 428 (1e-5) E-box(bHLH)/Promoter/Homer\n",
      "\t\t92 of 428 (1e-5) ZBTB33(Zf)/GM12878-ZBTB33-ChIP-Seq(GSE32465)/Homer\n",
      "\t\t93 of 428 (1e-5) Stat3(Stat)/mES-Stat3-ChIP-Seq(GSE11431)/Homer\n",
      "\t\t94 of 428 (1e-5) Foxo1(Forkhead)/RAW-Foxo1-ChIP-Seq(Fan_et_al.)/Homer\n",
      "\t\t95 of 428 (1e-5) STAT1(Stat)/HelaS3-STAT1-ChIP-Seq(GSE12782)/Homer\n",
      "\t\t96 of 428 (1e-5) bHLHE40(bHLH)/HepG2-BHLHE40-ChIP-Seq(GSE31477)/Homer\n",
      "\t\t97 of 428 (1e-4) STAT5(Stat)/mCD4+-Stat5-ChIP-Seq(GSE12346)/Homer\n",
      "\t\t98 of 428 (1e-4) STAT4(Stat)/CD4-Stat4-ChIP-Seq(GSE22104)/Homer\n",
      "\t\t99 of 428 (1e-4) CEBP:AP1(bZIP)/ThioMac-CEBPb-ChIP-Seq(GSE21512)/Homer\n",
      "\t\t100 of 428 (1e-4) c-Myc(bHLH)/LNCAP-cMyc-ChIP-Seq(Unpublished)/Homer\n",
      "\t\t101 of 428 (1e-4) RUNX-AML(Runt)/CD4+-PolII-ChIP-Seq(Barski_et_al.)/Homer\n",
      "\t\t102 of 428 (1e-4) Smad3(MAD)/NPC-Smad3-ChIP-Seq(GSE36673)/Homer\n",
      "\t\t103 of 428 (1e-4) ZNF711(Zf)/SHSY5Y-ZNF711-ChIP-Seq(GSE20673)/Homer\n",
      "\t\t104 of 428 (1e-4) Lhx3(Homeobox)/Neuron-Lhx3-ChIP-Seq(GSE31456)/Homer\n",
      "\t\t105 of 428 (1e-4) PRDM1(Zf)/Hela-PRDM1-ChIP-Seq(GSE31477)/Homer\n",
      "\t\t106 of 428 (1e-4) THRb(NR)/HepG2-THRb.Flag-ChIP-Seq(Encode)/Homer\n",
      "\t\t107 of 428 (1e-4) Srebp1a(bHLH)/HepG2-Srebp1a-ChIP-Seq(GSE31477)/Homer\n",
      "\t\t108 of 428 (1e-3) RFX(HTH)/K562-RFX3-ChIP-Seq(SRA012198)/Homer\n",
      "\t\t109 of 428 (1e-3) RUNX2(Runt)/PCa-RUNX2-ChIP-Seq(GSE33889)/Homer\n",
      "\t\t110 of 428 (1e-3) Tgif1(Homeobox)/mES-Tgif1-ChIP-Seq(GSE55404)/Homer\n",
      "\t\t111 of 428 (1e-3) Srebp2(bHLH)/HepG2-Srebp2-ChIP-Seq(GSE31477)/Homer\n",
      "\t\t112 of 428 (1e-3) Meis1(Homeobox)/MastCells-Meis1-ChIP-Seq(GSE48085)/Homer\n",
      "\t\t113 of 428 (1e-3) bHLHE41(bHLH)/proB-Bhlhe41-ChIP-Seq(GSE93764)/Homer\n",
      "\t\t114 of 428 (1e-3) T1ISRE(IRF)/ThioMac-Ifnb-Expression/Homer\n",
      "\t\t115 of 428 (1e-3) Egr2(Zf)/Thymocytes-Egr2-ChIP-Seq(GSE34254)/Homer\n",
      "\t\t116 of 428 (1e-3) Foxo3(Forkhead)/U2OS-Foxo3-ChIP-Seq(E-MTAB-2701)/Homer\n",
      "\t\t117 of 428 (1e-3) Max(bHLH)/K562-Max-ChIP-Seq(GSE31477)/Homer\n",
      "\t\t118 of 428 (1e-3) Rfx2(HTH)/LoVo-RFX2-ChIP-Seq(GSE49402)/Homer\n",
      "\t\t119 of 428 (1e-3) LXH9(Homeobox)/Hct116-LXH9.V5-ChIP-Seq(GSE116822)/Homer\n",
      "\t\t120 of 428 (1e-3) Pdx1(Homeobox)/Islet-Pdx1-ChIP-Seq(SRA008281)/Homer\n",
      "\t\t121 of 428 (1e-3) MNT(bHLH)/HepG2-MNT-ChIP-Seq(Encode)/Homer\n",
      "\t\t122 of 428 (1e-3) FOXK2(Forkhead)/U2OS-FOXK2-ChIP-Seq(E-MTAB-2204)/Homer\n",
      "\t\t123 of 428 (1e-3) RUNX1(Runt)/Jurkat-RUNX1-ChIP-Seq(GSE29180)/Homer\n",
      "\t\t124 of 428 (1e-3) Tgif2(Homeobox)/mES-Tgif2-ChIP-Seq(GSE55404)/Homer\n",
      "\t\t125 of 428 (1e-3) MafF(bZIP)/HepG2-MafF-ChIP-Seq(GSE31477)/Homer\n",
      "\t\t126 of 428 (1e-2) Arnt:Ahr(bHLH)/MCF7-Arnt-ChIP-Seq(Lo_et_al.)/Homer\n",
      "\t\t127 of 428 (1e-2) Brn1(POU,Homeobox)/NPC-Brn1-ChIP-Seq(GSE35496)/Homer\n",
      "\t\t128 of 428 (1e-2) Nkx3.1(Homeobox)/LNCaP-Nkx3.1-ChIP-Seq(GSE28264)/Homer\n",
      "\t\t129 of 428 (1e-2) n-Myc(bHLH)/mES-nMyc-ChIP-Seq(GSE11431)/Homer\n",
      "\t\t130 of 428 (1e-2) RUNX(Runt)/HPC7-Runx1-ChIP-Seq(GSE22178)/Homer\n",
      "\t\t131 of 428 (1e-2) BMAL1(bHLH)/Liver-Bmal1-ChIP-Seq(GSE39860)/Homer\n",
      "\t\t132 of 428 (1e-2) Hoxa13(Homeobox)/ChickenMSG-Hoxa13.Flag-ChIP-Seq(GSE86088)/Homer\n",
      "\t\t133 of 428 (1e-2) Oct4(POU,Homeobox)/mES-Oct4-ChIP-Seq(GSE11431)/Homer\n",
      "\t\t134 of 428 (1e-2) MYB(HTH)/ERMYB-Myb-ChIPSeq(GSE22095)/Homer\n",
      "\t\t135 of 428 (1e-2) FOXP1(Forkhead)/H9-FOXP1-ChIP-Seq(GSE31006)/Homer\n",
      "\t\t136 of 428 (1e-2) BMYB(HTH)/Hela-BMYB-ChIP-Seq(GSE27030)/Homer\n",
      "\t\t137 of 428 (1e-2) E2F4(E2F)/K562-E2F4-ChIP-Seq(GSE31477)/Homer\n",
      "\t\t138 of 428 (1e-2) Oct6(POU,Homeobox)/NPC-Pou3f1-ChIP-Seq(GSE35496)/Homer\n",
      "\t\t139 of 428 (1e-2) Atf4(bZIP)/MEF-Atf4-ChIP-Seq(GSE35681)/Homer\n",
      "\t\t140 of 428 (1e-2) Egr1(Zf)/K562-Egr1-ChIP-Seq(GSE32465)/Homer\n",
      "\t\t141 of 428 (1e-2) Hoxa10(Homeobox)/ChickenMSG-Hoxa10.Flag-ChIP-Seq(GSE86088)/Homer\n",
      "\t\t142 of 428 (1e-2) AMYB(HTH)/Testes-AMYB-ChIP-Seq(GSE44588)/Homer\n",
      "\t\t143 of 428 (1e-2) Foxf1(Forkhead)/Lung-Foxf1-ChIP-Seq(GSE77951)/Homer\n",
      "\t\t144 of 428 (1e-2) WT1(Zf)/Kidney-WT1-ChIP-Seq(GSE90016)/Homer\n",
      "\t\t145 of 428 (1e-2) Chop(bZIP)/MEF-Chop-ChIP-Seq(GSE35681)/Homer\n",
      "\t\t146 of 428 (1e-2) CArG(MADS)/PUER-Srf-ChIP-Seq(Sullivan_et_al.)/Homer\n",
      "\t\t147 of 428 (1e-2) FoxL2(Forkhead)/Ovary-FoxL2-ChIP-Seq(GSE60858)/Homer\n",
      "\t\t148 of 428 (1e-2) CEBP:CEBP(bZIP)/MEF-Chop-ChIP-Seq(GSE35681)/Homer\n",
      "\t\t149 of 428 (1e-2) p73(p53)/Trachea-p73-ChIP-Seq(PRJNA310161)/Homer\n",
      "\t\t150 of 428 (1e-2) Rfx5(HTH)/GM12878-Rfx5-ChIP-Seq(GSE31477)/Homer\n",
      "\t\t151 of 428 (1e-2) HINFP(Zf)/K562-HINFP.eGFP-ChIP-Seq(Encode)/Homer\n",
      "\t\t152 of 428 (1e-2) X-box(HTH)/NPC-H3K4me1-ChIP-Seq(GSE16256)/Homer\n",
      "\t\t153 of 428 (1e-2) ISRE(IRF)/ThioMac-LPS-Expression(GSE23622)/Homer\n",
      "\t----------------------------------------------------------\n",
      "\tDe novo motif finding (HOMER)\n",
      "\n",
      "\tNumber of Trial motifs (-T) set to 24 (from 10) to work well with 24 CPUs\n",
      "\tScanning input files...\n",
      "\tParsing sequences...\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\tTotal number of Oligos: 32896\n",
      "\tAutoadjustment for sequence coverage in background: 0.98x\n",
      "\n",
      "\tOligos: 32896 of 34497 max\n",
      "\tTree  : 67180 of 172485 max\n",
      "\tOptimizing memory usage...\n",
      "\tCache length = 11180\n",
      "\tUsing binomial scoring\n",
      "\n",
      "\tGlobal Optimization Phase: Looking for enriched oligos with up to 3 mismatches...\n",
      "\n",
      "\tScreening oligos 32896 (allowing 0 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t1.21% skipped, 98.79% checked (32498 of 32896), of those checked:\n",
      "\t\t1.21% not in target, 0.00% increased p-value, 0.00% high p-value\n",
      "\n",
      "\tScreening oligos 32896 (allowing 1 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t1.21% skipped, 98.79% checked (32498 of 32896), of those checked:\n",
      "\t\t0.00% not in target, 43.41% increased p-value, 58.93% high p-value\n",
      "\n",
      "\tScreening oligos 32896 (allowing 2 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t70.46% skipped, 29.54% checked (9718 of 32896), of those checked:\n",
      "\t\t0.00% not in target, 18.33% increased p-value, 0.00% high p-value\n",
      "\n",
      "\tScreening oligos 32896 (allowing 3 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t88.79% skipped, 11.21% checked (3688 of 32896), of those checked:\n",
      "\t\t0.00% not in target, 5.10% increased p-value, 0.00% high p-value\n",
      "\tReading input files...\n",
      "\t32625 total sequences read\n",
      "\tCache length = 11180\n",
      "\tUsing binomial scoring\n",
      "\n",
      "\tLocal Optimization Phase:\n",
      "\t1 of 25 Initial Sequence: ACTTCCTC... (-139.512)\n",
      "\t\tRound 1: -313.88 ACTTCCKS T:1974.0(43.02%),B:7502.9(23.87%),P:1e-136\n",
      "\t\tRound 2: -313.88 ACTTCCKS T:1974.0(43.02%),B:7502.9(23.87%),P:1e-136\n",
      "\t\t=Final=: -354.60 ACTTCCKS T:1433.0(40.83%),B:5883.4(21.04%),P:1e-153\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t2 of 25 Initial Sequence: GCCCCGCC... (-127.028)\n",
      "\t\tRound 1: -249.02 GCCCCGCC T:2056.0(44.34%),B:8617.1(26.89%),P:1e-108\n",
      "\t\tRound 2: -249.02 GCCCCGCC T:2056.0(44.34%),B:8617.1(26.89%),P:1e-108\n",
      "\t\t=Final=: -189.40 GCCCCGCC T:1134.0(32.31%),B:5220.0(18.67%),P:1e-82\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t3 of 25 Initial Sequence: GCCAATCA... (-87.274)\n",
      "\t\tRound 1: -204.71 GCCAATCA T:731.0(18.80%),B:2329.3(8.12%),P:1e-88\n",
      "\t\tRound 2: -204.71 GCCAATCA T:731.0(18.80%),B:2329.3(8.12%),P:1e-88\n",
      "\t\t=Final=: -163.13 GCCAATCA T:570.0(16.24%),B:2029.8(7.26%),P:1e-70\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t4 of 25 Initial Sequence: GCATGCGC... (-65.520)\n",
      "\t\tRound 1: -91.81 GCATGCGC T:1072.0(26.32%),B:5228.0(17.30%),P:1e-39\n",
      "\t\tRound 2: -91.81 GCATGCGC T:1072.0(26.32%),B:5228.0(17.30%),P:1e-39\n",
      "\t\t=Final=: -40.42 GCATGCGC T:488.0(13.90%),B:2620.7(9.37%),P:1e-17\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t5 of 25 Initial Sequence: ATGACGTC... (-62.584)\n",
      "\t\tRound 1: -95.98 ATGACGTC T:672.0(17.43%),B:2868.3(9.90%),P:1e-41\n",
      "\t\tRound 2: -105.86 RTSACGYM T:978.0(24.32%),B:4486.1(15.05%),P:1e-45\n",
      "\t\tRound 3: -122.36 RTNACGCV T:947.0(23.65%),B:4119.8(13.90%),P:1e-53\n",
      "\t\tRound 4: -122.36 RTNACGCV T:947.0(23.65%),B:4119.8(13.90%),P:1e-53\n",
      "\t\t=Final=: -113.62 RTNACGCV T:642.0(18.29%),B:2798.7(10.01%),P:1e-49\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t6 of 25 Initial Sequence: TCGCGAGA... (-34.041)\n",
      "\t\tRound 1: -40.30 TCGCGAGA T:133.0(3.72%),B:436.4(1.57%),P:1e-17\n",
      "\t\tRound 2: -42.97 TCGCGCTA T:143.0(3.99%),B:472.9(1.70%),P:1e-18\n",
      "\t\tRound 3: -42.97 TCGCGCTA T:143.0(3.99%),B:472.9(1.70%),P:1e-18\n",
      "\t\t=Final=: -23.88 TCGCGCTA T:98.0(2.79%),B:375.2(1.34%),P:1e-10\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t7 of 25 Initial Sequence: AAGAGGAA... (-33.814)\n",
      "\t\tRound 1: -51.29 AAGAGGAA T:159.0(4.43%),B:501.1(1.80%),P:1e-22\n",
      "\t\tRound 2: -51.29 AAGAGGAA T:159.0(4.43%),B:501.1(1.80%),P:1e-22\n",
      "\t\t=Final=: -50.90 AAGAGGAA T:214.0(6.10%),B:816.9(2.92%),P:1e-22\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t8 of 25 Initial Sequence: GGGGCGGG... (-33.101)\n",
      "\t\tRound 1: -100.84 GGGGCGGG T:1866.0(41.24%),B:9854.0(30.10%),P:1e-43\n",
      "\t\tRound 2: -100.84 GGGGCGGG T:1866.0(41.24%),B:9854.0(30.10%),P:1e-43\n",
      "\t\t=Final=: -96.25 GGGGCGGG T:1342.0(38.23%),B:7718.4(27.60%),P:1e-41\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t9 of 25 Initial Sequence: AAGATGGC... (-31.687)\n",
      "\t\tRound 1: -45.72 AAGATGGC T:243.0(6.69%),B:969.6(3.46%),P:1e-19\n",
      "\t\tRound 2: -45.72 AAGATGGC T:243.0(6.69%),B:969.6(3.46%),P:1e-19\n",
      "\t\t=Final=: -42.55 AAGATGGC T:226.0(6.44%),B:948.9(3.39%),P:1e-18\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t10 of 25 Initial Sequence: AACTACAA... (-23.945)\n",
      "\t\tRound 1: -66.27 AACTACAA T:174.0(4.84%),B:495.2(1.78%),P:1e-28\n",
      "\t\tRound 2: -66.27 AACTACAA T:174.0(4.84%),B:495.2(1.78%),P:1e-28\n",
      "\t\t=Final=: -52.30 AACTACAA T:151.0(4.30%),B:479.6(1.72%),P:1e-22\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t11 of 25 Initial Sequence: ACCAATAG... (-22.325)\n",
      "\t\tRound 1: -36.54 ACCAATAG T:143.0(3.99%),B:511.2(1.84%),P:1e-15\n",
      "\t\tRound 2: -36.54 ACCAATAG T:143.0(3.99%),B:511.2(1.84%),P:1e-15\n",
      "\t\t=Final=: -36.48 ACCAATAG T:138.0(3.93%),B:504.8(1.81%),P:1e-15\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t12 of 25 Initial Sequence: TTGCGTCA... (-17.357)\n",
      "\t\tRound 1: -36.21 TTGCGTCA T:303.0(8.27%),B:1409.4(4.99%),P:1e-15\n",
      "\t\tRound 2: -36.89 TTGCGTCA T:205.0(5.67%),B:840.4(3.01%),P:1e-16\n",
      "\t\tRound 3: -43.12 TWGCGTAA T:809.0(20.59%),B:4449.8(14.93%),P:1e-18\n",
      "\t\tRound 4: -43.12 TWGCGTAA T:809.0(20.59%),B:4449.8(14.93%),P:1e-18\n",
      "\t\t=Final=: -42.48 TWGCGTAA T:704.0(20.06%),B:4060.4(14.52%),P:1e-18\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t13 of 25 Initial Sequence: TGACGTCA... (-14.879)\n",
      "\t\tRound 1: -19.99 TGACGTCA T:278.0(7.62%),B:1483.6(5.25%),P:1e-8\n",
      "\t\tRound 2: -19.99 TGACGTCA T:278.0(7.62%),B:1483.6(5.25%),P:1e-8\n",
      "\t\t=Final=: -23.28 TGACGTCA T:270.0(7.69%),B:1435.1(5.13%),P:1e-10\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t14 of 25 Initial Sequence: TTAAAGGG... (-14.262)\n",
      "\t\tRound 1: -18.55 TTAAAGGG T:123.0(3.44%),B:544.3(1.96%),P:1e-8\n",
      "\t\tRound 2: -21.39 TYATAGGG T:83.0(2.34%),B:303.1(1.10%),P:1e-9\n",
      "\t\tRound 3: -21.39 TYATAGGG T:83.0(2.34%),B:303.1(1.10%),P:1e-9\n",
      "\t\t=Final=: -19.43 TYATAGGG T:78.0(2.22%),B:298.2(1.07%),P:1e-8\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t15 of 25 Initial Sequence: AAGCAGAA... (-12.790)\n",
      "\t\tRound 1: -15.40 AAGCAGAA T:99.0(2.78%),B:439.0(1.58%),P:1e-6\n",
      "\t\tRound 2: -17.37 AACCACAA T:480.0(12.78%),B:2871.5(9.91%),P:1e-7\n",
      "\t\tRound 3: -20.74 AACCACAA T:371.0(10.03%),B:2072.4(7.25%),P:1e-9\n",
      "\t\tRound 4: -20.74 AACCACAA T:371.0(10.03%),B:2072.4(7.25%),P:1e-9\n",
      "\t\t=Final=: -21.92 AACCACAA T:332.0(9.46%),B:1870.4(6.69%),P:1e-9\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t16 of 25 Initial Sequence: GTTTCCGG... (-12.294)\n",
      "\t\tRound 1: -20.65 GTTTCCGG T:144.0(4.02%),B:649.5(2.33%),P:1e-8\n",
      "\t\tRound 2: -20.65 GTTTCCGG T:144.0(4.02%),B:649.5(2.33%),P:1e-8\n",
      "\t\t=Final=: -22.37 GTTTCCGG T:139.0(3.96%),B:622.4(2.23%),P:1e-9\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t17 of 25 Initial Sequence: GGGGCGGA... (-11.518)\n",
      "\t\tRound 1: -6.87 GGGGCGGA T:282.0(7.72%),B:1820.8(6.40%),P:1e-2\n",
      "\t\tRound 2: -15.04 WGRGCGGA T:556.0(14.65%),B:3463.6(11.83%),P:1e-6\n",
      "\t\tRound 3: -15.04 WGRGCGGA T:556.0(14.65%),B:3463.6(11.83%),P:1e-6\n",
      "\t\t=Final=: -15.39 WGRGCGGA T:551.0(15.70%),B:3565.2(12.75%),P:1e-6\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t18 of 25 Initial Sequence: TTACGCTC... (-11.518)\n",
      "\t\tRound 1: -7.48 TTACGCTC T:21.0(0.60%),B:68.6(0.25%),P:1e-3\n",
      "\t\tRound 2: -10.94 GTACGCTC T:67.0(1.89%),B:298.9(1.08%),P:1e-4\n",
      "\t\tRound 3: -12.75 TTACGGTC T:59.0(1.67%),B:235.6(0.85%),P:1e-5\n",
      "\t\tRound 4: -12.75 TTACGGTC T:59.0(1.67%),B:235.6(0.85%),P:1e-5\n",
      "\t\t=Final=: -14.21 TTACGGTC T:59.0(1.68%),B:232.9(0.83%),P:1e-6\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t19 of 25 Initial Sequence: GGGAGGAG... (-11.099)\n",
      "\t\tRound 1: -14.46 GGGAGGAG T:384.0(10.36%),B:2296.1(8.01%),P:1e-6\n",
      "\t\tRound 2: -14.75 GGGMGGAG T:386.0(10.42%),B:2303.3(8.03%),P:1e-6\n",
      "\t\tRound 3: -14.75 GGGMGGAG T:386.0(10.42%),B:2303.3(8.03%),P:1e-6\n",
      "\t\t=Final=: -14.34 GGGMGGAG T:351.0(10.00%),B:2156.3(7.71%),P:1e-6\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t20 of 25 Initial Sequence: GGAAGTGT... (-9.801)\n",
      "\t\tRound 1: -14.51 AGAAGTGT T:102.0(2.86%),B:465.8(1.68%),P:1e-6\n",
      "\t\tRound 2: -14.51 AGAAGTGT T:102.0(2.86%),B:465.8(1.68%),P:1e-6\n",
      "\t\t=Final=: -15.12 AGAAGTGT T:100.0(2.85%),B:462.3(1.65%),P:1e-6\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\tRemaining seeds don't look promising (After initial 5 motifs, logp -8.968 > -9.545)\n",
      "\n",
      "\tFinalizing Enrichment Statistics (new in v3.4)\n",
      "\tReading input files...\n",
      "\t32625 total sequences read\n",
      "\tCache length = 11180\n",
      "\tUsing binomial scoring\n",
      "\tChecking enrichment of 20 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tOutput in file: getDiff_Nathan_BMDM_K27ac_GW_gained_20_FC1_5_p0001_GC_matched.txt/homerMotifs.motifs8\n",
      "\n",
      "\n",
      "\tNumber of Trial motifs (-T) set to 24 (from 10) to work well with 24 CPUs\n",
      "\tScanning input files...\n",
      "\tParsing sequences...\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\tTotal number of Oligos: 485727\n",
      "\tAutoadjustment for sequence coverage in background: 0.98x\n",
      "\n",
      "\tOligos: 485727 of 551887 max\n",
      "\tTree  : 1007056 of 2759435 max\n",
      "\tOptimizing memory usage...\n",
      "\tCache length = 11180\n",
      "\tUsing binomial scoring\n",
      "\n",
      "\tGlobal Optimization Phase: Looking for enriched oligos with up to 3 mismatches...\n",
      "\n",
      "\tScreening oligos 485727 (allowing 0 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t45.16% skipped, 54.84% checked (266396 of 485727), of those checked:\n",
      "\t\t45.16% not in target, 0.00% increased p-value, 0.00% high p-value\n",
      "\n",
      "\tScreening oligos 485727 (allowing 1 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t45.16% skipped, 54.84% checked (266396 of 485727), of those checked:\n",
      "\t\t0.00% not in target, 30.16% increased p-value, 28.58% high p-value\n",
      "\n",
      "\tScreening oligos 485727 (allowing 2 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t86.36% skipped, 13.64% checked (66257 of 485727), of those checked:\n",
      "\t\t0.00% not in target, 6.85% increased p-value, 0.00% high p-value\n",
      "\n",
      "\tScreening oligos 485727 (allowing 3 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t93.21% skipped, 6.79% checked (32973 of 485727), of those checked:\n",
      "\t\t0.00% not in target, 5.02% increased p-value, 0.00% high p-value\n",
      "\tReading input files...\n",
      "\t32625 total sequences read\n",
      "\tCache length = 11180\n",
      "\tUsing binomial scoring\n",
      "\n",
      "\tLocal Optimization Phase:\n",
      "\t1 of 25 Initial Sequence: ACTTCCTCTT... (-164.503)\n",
      "\t\tRound 1: -296.81 ACTTCCTCTT T:1127.0(27.47%),B:3597.9(12.26%),P:1e-128\n",
      "\t\tRound 2: -357.72 ACTTCCTCYT T:1775.0(39.70%),B:6158.7(20.06%),P:1e-155\n",
      "\t\tRound 3: -362.24 ACTTCCKBTT T:1637.0(37.28%),B:5485.6(18.08%),P:1e-157\n",
      "\t\tRound 4: -363.09 ACTTCCTYTT T:1725.0(38.83%),B:5880.6(19.25%),P:1e-157\n",
      "\t\tRound 5: -363.09 ACTTCCTYTT T:1725.0(38.83%),B:5880.6(19.25%),P:1e-157\n",
      "\t\t=Final=: -386.40 ACTTCCTYTT T:1291.0(36.78%),B:4809.9(17.20%),P:1e-167\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t2 of 25 Initial Sequence: GGGCGGGGCC... (-155.705)\n",
      "\t\tRound 1: -336.95 GGGCGGGGCC T:2037.0(44.03%),B:7578.4(24.09%),P:1e-146\n",
      "\t\tRound 2: -364.13 RGGCGGGGYY T:2024.0(43.83%),B:7270.2(23.23%),P:1e-158\n",
      "\t\tRound 3: -364.13 RGGCGGGGYY T:2024.0(43.83%),B:7270.2(23.23%),P:1e-158\n",
      "\t\t=Final=: -292.90 RGGCGGGGYY T:1255.0(35.75%),B:5182.9(18.54%),P:1e-127\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t3 of 25 Initial Sequence: CGCATGCGCA... (-101.682)\n",
      "\t\tRound 1: -141.86 CGCATGCGCA T:797.0(20.32%),B:3113.9(10.70%),P:1e-61\n",
      "\t\tRound 2: -141.86 CGCATGCGCA T:797.0(20.32%),B:3113.9(10.70%),P:1e-61\n",
      "\t\t=Final=: -71.36 CGCATGCGCA T:335.0(9.54%),B:1340.2(4.79%),P:1e-30\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t4 of 25 Initial Sequence: CATGACGTCA... (-73.443)\n",
      "\t\tRound 1: -126.55 CATGACGTCA T:589.0(15.45%),B:2158.8(7.55%),P:1e-54\n",
      "\t\tRound 2: -138.06 CATGACGTCA T:730.0(18.78%),B:2795.8(9.66%),P:1e-59\n",
      "\t\tRound 3: -138.06 CATGACGTCA T:730.0(18.78%),B:2795.8(9.66%),P:1e-59\n",
      "\t\t=Final=: -83.83 CATGACGTCA T:434.0(12.36%),B:1811.7(6.48%),P:1e-36\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t5 of 25 Initial Sequence: TGATTGGCTA... (-63.299)\n",
      "\t\tRound 1: -154.72 TGATTGGCTA T:634.0(16.53%),B:2185.8(7.64%),P:1e-67\n",
      "\t\tRound 2: -225.64 TSATTGGTCR T:820.0(20.84%),B:2620.2(9.09%),P:1e-97\n",
      "\t\tRound 3: -225.64 TSATTGGTCR T:820.0(20.84%),B:2620.2(9.09%),P:1e-97\n",
      "\t\t=Final=: -158.61 TSATTGGTCR T:511.0(14.56%),B:1740.5(6.22%),P:1e-68\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t6 of 25 Initial Sequence: ACTACAATTC... (-56.663)\n",
      "\t\tRound 1: -88.71 ACTACAATTC T:167.0(4.65%),B:381.9(1.38%),P:1e-38\n",
      "\t\tRound 2: -88.71 ACTACAATTC T:167.0(4.65%),B:381.9(1.38%),P:1e-38\n",
      "\t\t=Final=: -79.24 ACTACAATTC T:122.0(3.48%),B:250.5(0.90%),P:1e-34\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t7 of 25 Initial Sequence: TGCTGAGTCA... (-41.024)\n",
      "\t\tRound 1: -58.96 TGCTGAGTCA T:227.0(6.26%),B:789.0(2.82%),P:1e-25\n",
      "\t\tRound 2: -83.21 TGSTGAGTCA T:372.0(10.06%),B:1366.2(4.85%),P:1e-36\n",
      "\t\tRound 3: -83.21 TGSTGAGTCA T:372.0(10.06%),B:1366.2(4.85%),P:1e-36\n",
      "\t\t=Final=: -69.62 TGSTGAGTCA T:235.0(6.70%),B:808.5(2.89%),P:1e-30\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t8 of 25 Initial Sequence: CTCGCGAGAT... (-30.531)\n",
      "\t\tRound 1: -44.00 CTCGCGAGAT T:146.0(4.07%),B:481.9(1.73%),P:1e-19\n",
      "\t\tRound 2: -44.00 CTCGCGAGAT T:146.0(4.07%),B:481.9(1.73%),P:1e-19\n",
      "\t\t=Final=: -23.22 CTCGCGAGAT T:86.0(2.45%),B:315.2(1.13%),P:1e-10\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t9 of 25 Initial Sequence: AAGATGGCGG... (-28.440)\n",
      "\t\tRound 1: -45.73 AAGATGGCGG T:139.0(3.88%),B:436.0(1.57%),P:1e-19\n",
      "\t\tRound 2: -45.73 AAGATGGCGG T:139.0(3.88%),B:436.0(1.57%),P:1e-19\n",
      "\t\t=Final=: -42.70 AAGATGGCGG T:165.0(4.70%),B:607.6(2.17%),P:1e-18\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t10 of 25 Initial Sequence: CTGTCAATCA... (-28.266)\n",
      "\t\tRound 1: -36.41 CTGTCAATCA T:105.0(2.95%),B:322.1(1.16%),P:1e-15\n",
      "\t\tRound 2: -36.41 CTGTCAATCA T:105.0(2.95%),B:322.1(1.16%),P:1e-15\n",
      "\t\t=Final=: -34.80 CTGTCAATCA T:100.0(2.85%),B:319.3(1.14%),P:1e-15\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t11 of 25 Initial Sequence: GTTCCGCCCT... (-21.730)\n",
      "\t\tRound 1: -33.21 GTTCCGCCCT T:1163.0(28.21%),B:7032.9(22.56%),P:1e-14\n",
      "\t\tRound 2: -33.21 GTTCCGCCCT T:1163.0(28.21%),B:7032.9(22.56%),P:1e-14\n",
      "\t\t=Final=: -37.18 GTTCCGCCCT T:556.0(15.84%),B:3131.0(11.20%),P:1e-16\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t12 of 25 Initial Sequence: AGTAACCCCG... (-19.044)\n",
      "\t\tRound 1: -19.04 AGTAACCCCG T:7.0(0.20%),B:2.1(0.01%),P:1e-8\n",
      "\t\tRound 2: -19.04 AGTAACCCCG T:7.0(0.20%),B:2.1(0.01%),P:1e-8\n",
      "\t\t=Final=: -15.09 AGTAACCCCG T:6.0(0.17%),B:2.1(0.01%),P:1e-6\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t13 of 25 Initial Sequence: CGCCAATAGG... (-17.350)\n",
      "\t\tRound 1: -27.02 CGCCAATAGG T:234.0(6.45%),B:1107.8(3.95%),P:1e-11\n",
      "\t\tRound 2: -27.62 CGCCAATAGG T:179.0(4.97%),B:777.8(2.79%),P:1e-11\n",
      "\t\tRound 3: -28.59 CGCCAATAGG T:213.0(5.89%),B:963.8(3.44%),P:1e-12\n",
      "\t\tRound 4: -29.20 CGCCAATAGG T:175.0(4.86%),B:738.5(2.65%),P:1e-12\n",
      "\t\tRound 5: -31.07 CGCCAATAGG T:223.0(6.16%),B:998.7(3.56%),P:1e-13\n",
      "\t\tRound 6: -35.14 MGCCAATWGG T:318.0(8.66%),B:1512.9(5.35%),P:1e-15\n",
      "\t\tRound 7: -35.14 MGCCAATWGG T:318.0(8.66%),B:1512.9(5.35%),P:1e-15\n",
      "\t\t=Final=: -35.49 MGCCAATWGG T:291.0(8.29%),B:1410.1(5.04%),P:1e-15\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t14 of 25 Initial Sequence: GTCACGTGAC... (-17.126)\n",
      "\t\tRound 1: -23.32 GTCACGTGAC T:70.0(1.97%),B:224.8(0.81%),P:1e-10\n",
      "\t\tRound 2: -24.54 GTCACGTGAC T:75.0(2.11%),B:243.1(0.88%),P:1e-10\n",
      "\t\tRound 3: -24.54 GTCACGTGAC T:75.0(2.11%),B:243.1(0.88%),P:1e-10\n",
      "\t\t=Final=: -24.00 GTCACGTGAC T:71.0(2.02%),B:233.7(0.84%),P:1e-10\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t15 of 25 Initial Sequence: GGGGCGGGTA... (-16.156)\n",
      "\t\tRound 1: -28.87 GGGGCGGGTA T:317.0(8.64%),B:1590.8(5.62%),P:1e-12\n",
      "\t\tRound 2: -40.75 GGGGCGGGKM T:713.0(18.39%),B:3882.1(13.17%),P:1e-17\n",
      "\t\tRound 3: -55.61 GGGGCGKGKH T:576.0(15.14%),B:2789.7(9.64%),P:1e-24\n",
      "\t\tRound 4: -59.64 KGGGCGKDKC T:739.0(18.99%),B:3716.1(12.64%),P:1e-25\n",
      "\t\tRound 5: -62.90 KGGGMGGRGC T:849.0(21.49%),B:4339.0(14.59%),P:1e-27\n",
      "\t\tRound 6: -62.90 KGGGMGGRGC T:849.0(21.49%),B:4339.0(14.59%),P:1e-27\n",
      "\t\t=Final=: -64.64 KGGGMGGRGC T:733.0(20.88%),B:3914.5(14.00%),P:1e-28\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t16 of 25 Initial Sequence: TTCCTTCCTC... (-15.732)\n",
      "\t\tRound 1: -18.78 TTCCTTCCTC T:25.0(0.71%),B:45.8(0.16%),P:1e-8\n",
      "\t\tRound 2: -18.78 TTCCTTCCTC T:25.0(0.71%),B:45.8(0.16%),P:1e-8\n",
      "\t\t=Final=: -20.18 TTCCTTCCTC T:25.0(0.71%),B:45.8(0.16%),P:1e-8\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t17 of 25 Initial Sequence: TTCCTCTTTC... (-15.448)\n",
      "\t\tRound 1: -26.05 TTCCTCTTTC T:40.0(1.13%),B:81.7(0.29%),P:1e-11\n",
      "\t\tRound 2: -26.05 TTCCTCTTTC T:40.0(1.13%),B:81.7(0.29%),P:1e-11\n",
      "\t\t=Final=: -28.71 TTCCTCTTTC T:40.0(1.14%),B:78.8(0.28%),P:1e-12\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t18 of 25 Initial Sequence: ACCAATGAGA... (-15.189)\n",
      "\t\tRound 1: -9.25 ACCAATGAGA T:76.0(2.14%),B:372.1(1.34%),P:1e-4\n",
      "\t\tRound 2: -13.58 VCCAATGAGA T:42.0(1.19%),B:139.5(0.50%),P:1e-5\n",
      "\t\tRound 3: -14.17 GCCARTSAGA T:26.0(0.74%),B:62.5(0.23%),P:1e-6\n",
      "\t\tRound 4: -15.24 GCCAATCAGA T:22.0(0.62%),B:43.9(0.16%),P:1e-6\n",
      "\t\tRound 5: -16.04 ACCAATCAGA T:26.0(0.74%),B:57.0(0.20%),P:1e-6\n",
      "\t\tRound 6: -16.04 ACCAATCAGA T:26.0(0.74%),B:57.0(0.20%),P:1e-6\n",
      "\t\t=Final=: -15.37 ACCAATCAGA T:25.0(0.71%),B:58.4(0.21%),P:1e-6\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t19 of 25 Initial Sequence: AAACTGAAAC... (-15.189)\n",
      "\t\tRound 1: -22.30 AAACTGAAAC T:11.0(0.31%),B:5.8(0.02%),P:1e-9\n",
      "\t\tRound 2: -31.45 AAACTGACGC T:10.0(0.28%),B:2.9(0.01%),P:1e-13\n",
      "\t\tRound 3: -31.45 AAACTGACGC T:10.0(0.28%),B:2.9(0.01%),P:1e-13\n",
      "\t\t=Final=: -29.17 AAACTGACGC T:10.0(0.28%),B:2.9(0.01%),P:1e-12\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t20 of 25 Initial Sequence: CTTAAAGGGA... (-15.071)\n",
      "\t\tRound 1: -28.98 CTTAAAGGGA T:74.0(2.09%),B:215.7(0.78%),P:1e-12\n",
      "\t\tRound 2: -28.98 CTTAAAGGGA T:74.0(2.09%),B:215.7(0.78%),P:1e-12\n",
      "\t\t=Final=: -28.37 CTTAAAGGGA T:71.0(2.02%),B:211.2(0.76%),P:1e-12\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t21 of 25 Initial Sequence: ACTTCCGGTG... (-14.027)\n",
      "\t\tRound 1: -23.43 ACTTCCGGTG T:107.0(3.00%),B:414.1(1.49%),P:1e-10\n",
      "\t\tRound 2: -23.51 AYTTCCGGYG T:167.0(4.65%),B:753.9(2.70%),P:1e-10\n",
      "\t\tRound 3: -29.18 AYTTCCGSYG T:294.0(8.04%),B:1445.5(5.12%),P:1e-12\n",
      "\t\tRound 4: -34.57 AYTTCCGGTG T:309.0(8.43%),B:1461.0(5.17%),P:1e-15\n",
      "\t\tRound 5: -39.29 ABTTCCGGTR T:275.0(7.54%),B:1210.7(4.30%),P:1e-17\n",
      "\t\tRound 6: -39.29 ABTTCCGGTR T:275.0(7.54%),B:1210.7(4.30%),P:1e-17\n",
      "\t\t=Final=: -38.05 ABTTCCGGTR T:257.0(7.32%),B:1173.1(4.20%),P:1e-16\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t22 of 25 Initial Sequence: TCTGCCCGCC... (-13.206)\n",
      "\t\tRound 1: -16.25 TCTGCCCGCC T:42.0(1.19%),B:125.5(0.45%),P:1e-7\n",
      "\t\tRound 2: -16.25 TCTGCCCGCC T:42.0(1.19%),B:125.5(0.45%),P:1e-7\n",
      "\t\t=Final=: -17.48 TCTGCCCGCC T:42.0(1.20%),B:125.5(0.45%),P:1e-7\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t23 of 25 Initial Sequence: AGCGCCACCT... (-12.788)\n",
      "\t\tRound 1: -20.32 AGCGCCACCT T:69.0(1.95%),B:237.1(0.86%),P:1e-8\n",
      "\t\tRound 2: -20.32 AGCGCCACCT T:69.0(1.95%),B:237.1(0.86%),P:1e-8\n",
      "\t\t=Final=: -22.02 AGCGCCACCT T:69.0(1.97%),B:235.0(0.84%),P:1e-9\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t24 of 25 Initial Sequence: ATTTCCTGAT... (-12.686)\n",
      "\t\tRound 1: -22.10 ATTTCCTGAT T:138.0(3.86%),B:598.1(2.15%),P:1e-9\n",
      "\t\tRound 2: -22.10 ATTTCCTGAT T:138.0(3.86%),B:598.1(2.15%),P:1e-9\n",
      "\t\t=Final=: -23.84 ATTTCCTGAT T:95.0(2.71%),B:359.0(1.28%),P:1e-10\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t25 of 25 Initial Sequence: GTTACGCAAT... (-12.742)\n",
      "\t\tRound 1: -30.46 GTTACGCAAT T:90.0(2.53%),B:280.0(1.01%),P:1e-13\n",
      "\t\tRound 2: -30.46 GTTACGCAAT T:90.0(2.53%),B:280.0(1.01%),P:1e-13\n",
      "\t\t=Final=: -32.12 GTTACGCAAT T:89.0(2.54%),B:279.2(1.00%),P:1e-13\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\n",
      "\tFinalizing Enrichment Statistics (new in v3.4)\n",
      "\tReading input files...\n",
      "\t32625 total sequences read\n",
      "\tCache length = 11180\n",
      "\tUsing binomial scoring\n",
      "\tChecking enrichment of 25 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tOutput in file: getDiff_Nathan_BMDM_K27ac_GW_gained_20_FC1_5_p0001_GC_matched.txt/homerMotifs.motifs10\n",
      "\n",
      "\n",
      "\tNumber of Trial motifs (-T) set to 24 (from 10) to work well with 24 CPUs\n",
      "\t-blen automatically set to 2\n",
      "\tScanning input files...\n",
      "\tParsing sequences...\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\tTotal number of Oligos: 2746823\n",
      "\tAutoadjustment for sequence coverage in background: 0.98x\n",
      "\n",
      "\tOligos: 2746823 of 6198750 max\n",
      "\tTree  : 6507372 of 30993750 max\n",
      "\tOptimizing memory usage...\n",
      "\tCache length = 11180\n",
      "\tUsing binomial scoring\n",
      "\n",
      "\tGlobal Optimization Phase: Looking for enriched oligos with up to 3 mismatches...\n",
      "\n",
      "\tScreening oligos 2746823 (allowing 0 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t80.61% skipped, 19.39% checked (532539 of 2746823), of those checked:\n",
      "\t\t80.61% not in target, 0.00% increased p-value, 0.00% high p-value\n",
      "\n",
      "\tScreening oligos 2746823 (allowing 1 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t80.61% skipped, 19.39% checked (532539 of 2746823), of those checked:\n",
      "\t\t0.00% not in target, 15.56% increased p-value, 2.18% high p-value\n",
      "\n",
      "\tScreening oligos 2746823 (allowing 2 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t96.62% skipped, 3.38% checked (92814 of 2746823), of those checked:\n",
      "\t\t0.00% not in target, 1.90% increased p-value, 0.00% high p-value\n",
      "\n",
      "\tScreening oligos 2746823 (allowing 3 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t98.52% skipped, 1.48% checked (40523 of 2746823), of those checked:\n",
      "\t\t0.00% not in target, 0.61% increased p-value, 0.00% high p-value\n",
      "\tReading input files...\n",
      "\t32625 total sequences read\n",
      "\tCache length = 11180\n",
      "\tUsing binomial scoring\n",
      "\n",
      "\tLocal Optimization Phase:\n",
      "\t1 of 25 Initial Sequence: AAAGAGGAAGTG... (-167.675)\n",
      "\t\tRound 1: -264.83 AAAGAGGAAGTG T:770.0(19.70%),B:2180.4(7.63%),P:1e-115\n",
      "\t\tRound 2: -305.15 AAAGAGGAAGTG T:957.0(23.87%),B:2784.0(9.63%),P:1e-132\n",
      "\t\tRound 3: -327.05 AAAGAGGAAGTG T:1630.0(37.15%),B:5724.0(18.80%),P:1e-142\n",
      "\t\tRound 4: -330.90 NARVAGGAAGYV T:2481.0(50.68%),B:9799.9(29.99%),P:1e-143\n",
      "\t\tRound 5: -330.90 NARVAGGAAGYV T:2481.0(50.68%),B:9799.9(29.99%),P:1e-143\n",
      "\t\t=Final=: -342.70 NARVAGGAAGYV T:1285.0(36.61%),B:5037.1(18.01%),P:1e-148\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t2 of 25 Initial Sequence: AGGCCCCGCCCA... (-158.528)\n",
      "\t\tRound 1: -284.44 AGGCCCCGCCCM T:1135.0(27.63%),B:3707.7(12.62%),P:1e-123\n",
      "\t\tRound 2: -347.61 AGGCCCCGCCCH T:1969.0(42.94%),B:7155.2(22.92%),P:1e-150\n",
      "\t\tRound 3: -414.18 AGGCCCCGCCCC T:1764.0(39.51%),B:5680.5(18.67%),P:1e-179\n",
      "\t\tRound 4: -414.18 AGGCCCCGCCCC T:1764.0(39.51%),B:5680.5(18.67%),P:1e-179\n",
      "\t\t=Final=: -332.00 AGGCCCCGCCCC T:1143.0(32.56%),B:4252.2(15.21%),P:1e-144\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t3 of 25 Initial Sequence: ATTGCGCATGCG... (-105.128)\n",
      "\t\tRound 1: -126.39 ATTGCGCATGCG T:601.0(15.74%),B:2219.5(7.76%),P:1e-54\n",
      "\t\tRound 2: -126.39 ATTGCGCATGCG T:601.0(15.74%),B:2219.5(7.76%),P:1e-54\n",
      "\t\t=Final=: -80.25 ATTGCGCATGCG T:354.0(10.09%),B:1381.0(4.94%),P:1e-34\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t4 of 25 Initial Sequence: ACTACAATTCCC... (-87.799)\n",
      "\t\tRound 1: -125.26 ACTACAATTCCC T:159.0(4.43%),B:256.5(0.93%),P:1e-54\n",
      "\t\tRound 2: -125.26 ACTACAATTCCC T:159.0(4.43%),B:256.5(0.93%),P:1e-54\n",
      "\t\t=Final=: -106.33 ACTACAATTCCC T:132.0(3.76%),B:221.9(0.79%),P:1e-46\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t5 of 25 Initial Sequence: ACGTCACGTGAC... (-70.589)\n",
      "\t\tRound 1: -71.02 ACGTCACGTGAC T:457.0(12.21%),B:1920.7(6.75%),P:1e-30\n",
      "\t\tRound 2: -71.02 ACGTCACGTGAC T:457.0(12.21%),B:1920.7(6.75%),P:1e-30\n",
      "\t\t=Final=: -35.45 ACGTCACGTGAC T:264.0(7.52%),B:1243.8(4.45%),P:1e-15\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t6 of 25 Initial Sequence: TAACCAATCGGA... (-61.532)\n",
      "\t\tRound 1: -160.21 TAACCAATCGGA T:692.0(17.90%),B:2432.4(8.47%),P:1e-69\n",
      "\t\tRound 2: -194.76 BRRCCAATMRGA T:537.0(14.19%),B:1508.5(5.34%),P:1e-84\n",
      "\t\tRound 3: -201.40 BARCCAATGRGM T:648.0(16.86%),B:1965.5(6.90%),P:1e-87\n",
      "\t\tRound 4: -214.85 YVRCCAATVAVV T:658.0(17.10%),B:1944.0(6.83%),P:1e-93\n",
      "\t\tRound 5: -214.85 YVRCCAATVAVV T:658.0(17.10%),B:1944.0(6.83%),P:1e-93\n",
      "\t\t=Final=: -157.98 YVRCCAATVAVV T:505.0(14.39%),B:1712.8(6.13%),P:1e-68\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t7 of 25 Initial Sequence: GCCACGCCCCCT... (-49.447)\n",
      "\t\tRound 1: -11.36 GCCACGCCCCCT T:95.0(2.67%),B:458.5(1.65%),P:1e-4\n",
      "\t\tRound 2: -16.43 GCCACGCCCCCT T:61.0(1.72%),B:219.1(0.79%),P:1e-7\n",
      "\t\tRound 3: -16.43 GCCACGCCCCCT T:61.0(1.72%),B:219.1(0.79%),P:1e-7\n",
      "\t\t=Final=: -17.99 GCCACGCCCCCT T:61.0(1.74%),B:217.4(0.78%),P:1e-7\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t8 of 25 Initial Sequence: CAATATGGCGGC... (-41.960)\n",
      "\t\tRound 1: -51.35 CAABATGGCGGC T:188.0(5.22%),B:644.7(2.32%),P:1e-22\n",
      "\t\tRound 2: -52.96 CAAGATGGCGGC T:158.0(4.40%),B:487.7(1.76%),P:1e-23\n",
      "\t\tRound 3: -58.70 CAAGATGGCGGC T:201.0(5.57%),B:662.7(2.38%),P:1e-25\n",
      "\t\tRound 4: -60.13 CAAGATGGCGKC T:203.0(5.62%),B:664.8(2.39%),P:1e-26\n",
      "\t\tRound 5: -60.13 CAAGATGGCGKC T:203.0(5.62%),B:664.8(2.39%),P:1e-26\n",
      "\t\t=Final=: -57.28 CAAGATGGCGKC T:191.0(5.44%),B:653.0(2.34%),P:1e-24\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t9 of 25 Initial Sequence: TCTCGCGAGATT... (-36.543)\n",
      "\t\tRound 1: -48.76 TCTCGCGAGATT T:97.0(2.73%),B:233.7(0.84%),P:1e-21\n",
      "\t\tRound 2: -48.76 TCTCGCGAGATT T:97.0(2.73%),B:233.7(0.84%),P:1e-21\n",
      "\t\t=Final=: -24.15 TCTCGCGAGATT T:52.0(1.48%),B:142.4(0.51%),P:1e-10\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t10 of 25 Initial Sequence: ATGACGTCACAC... (-31.393)\n",
      "\t\tRound 1: -49.35 ATGACGTCACAC T:169.0(4.70%),B:562.6(2.02%),P:1e-21\n",
      "\t\tRound 2: -49.35 ATGACGTCACAC T:169.0(4.70%),B:562.6(2.02%),P:1e-21\n",
      "\t\t=Final=: -44.44 ATGACGTCACAC T:314.0(8.95%),B:1454.0(5.20%),P:1e-19\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t11 of 25 Initial Sequence: CCACTAGAGGGC... (-31.702)\n",
      "\t\tRound 1: -51.83 CCACTAGAGGGC T:101.0(2.84%),B:240.1(0.87%),P:1e-22\n",
      "\t\tRound 2: -67.34 CCACTAGRGGGC T:189.0(5.24%),B:561.5(2.02%),P:1e-29\n",
      "\t\tRound 3: -68.99 CCACTAGRGGGC T:116.0(3.25%),B:246.8(0.89%),P:1e-29\n",
      "\t\tRound 4: -68.99 CCACTAGRGGGC T:116.0(3.25%),B:246.8(0.89%),P:1e-29\n",
      "\t\t=Final=: -70.43 CCACTAGRGGGC T:114.0(3.25%),B:244.6(0.87%),P:1e-30\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t12 of 25 Initial Sequence: TGCGCAGGCGCG... (-26.912)\n",
      "\t\tRound 1: -10.54 TGCGCAGGCGCG T:94.0(2.64%),B:462.1(1.67%),P:1e-4\n",
      "\t\tRound 2: -19.43 TGCGCAGGCGCG T:66.0(1.86%),B:226.6(0.82%),P:1e-8\n",
      "\t\tRound 3: -19.43 TGCGCAGGCGCG T:66.0(1.86%),B:226.6(0.82%),P:1e-8\n",
      "\t\t=Final=: -19.74 TGCGCAGGCGCG T:65.0(1.85%),B:227.3(0.81%),P:1e-8\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t13 of 25 Initial Sequence: CGTGATTGACAG... (-26.554)\n",
      "\t\tRound 1: -38.01 CGTGATTGACAG T:57.0(1.61%),B:112.8(0.41%),P:1e-16\n",
      "\t\tRound 2: -38.01 CGTGATTGACAG T:57.0(1.61%),B:112.8(0.41%),P:1e-16\n",
      "\t\t=Final=: -35.57 CGTGATTGACAG T:54.0(1.54%),B:112.8(0.40%),P:1e-15\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t14 of 25 Initial Sequence: GATGACTCAGCA... (-24.195)\n",
      "\t\tRound 1: -48.98 NNTGACTCAGNN T:251.0(6.90%),B:988.5(3.53%),P:1e-21\n",
      "\t\tRound 2: -62.76 BVTGACTCAGYV T:369.0(9.98%),B:1512.7(5.35%),P:1e-27\n",
      "\t\tRound 3: -81.30 SVTGACTCAGCR T:453.0(12.11%),B:1808.5(6.37%),P:1e-35\n",
      "\t\tRound 4: -81.30 SVTGACTCAGCR T:453.0(12.11%),B:1808.5(6.37%),P:1e-35\n",
      "\t\t=Final=: -61.09 SVTGACTCAGCR T:342.0(9.74%),B:1464.4(5.24%),P:1e-26\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t15 of 25 Initial Sequence: TGACCACAGAAA... (-24.147)\n",
      "\t\tRound 1: -35.25 TGACCACAGAAA T:16.0(0.45%),B:6.6(0.02%),P:1e-15\n",
      "\t\tRound 2: -35.25 TGACCACAGAAA T:16.0(0.45%),B:6.6(0.02%),P:1e-15\n",
      "\t\t=Final=: -35.95 TGACCACAGAAA T:16.0(0.46%),B:6.6(0.02%),P:1e-15\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t16 of 25 Initial Sequence: GCCACGCCCCCG... (-23.049)\n",
      "\t\tRound 1: -4.97 SNCACGCCCCNN T:8.0(0.23%),B:18.8(0.07%),P:1e-2\n",
      "\t\tRound 2: -4.97 SNCACGCCCCNN T:8.0(0.23%),B:18.8(0.07%),P:1e-2\n",
      "\t\t=Final=: -5.74 SNCACGCCCCNN T:8.0(0.23%),B:19.4(0.07%),P:1e-2\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t17 of 25 Initial Sequence: ACCCCGCCCCTC... (-23.049)\n",
      "\t\tRound 1: -6.36 ACCCCGCCCCTC T:193.0(5.35%),B:1203.6(4.28%),P:1e-2\n",
      "\t\tRound 2: -11.74 ACCCCGCCCCTC T:73.0(2.06%),B:325.4(1.18%),P:1e-5\n",
      "\t\tRound 3: -14.10 ACCCCGCCCCTC T:55.0(1.55%),B:203.4(0.74%),P:1e-6\n",
      "\t\tRound 4: -14.30 ACCCCKCCCCTC T:36.0(1.02%),B:106.1(0.38%),P:1e-6\n",
      "\t\tRound 5: -14.30 ACCCCKCCCCTC T:36.0(1.02%),B:106.1(0.38%),P:1e-6\n",
      "\t\t=Final=: -15.34 ACCCCKCCCCTC T:35.0(1.00%),B:102.8(0.37%),P:1e-6\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t18 of 25 Initial Sequence: ATTTGAGGAAAT... (-23.049)\n",
      "\t\tRound 1: -27.19 ATTTGAGGAAAT T:9.0(0.26%),B:2.1(0.01%),P:1e-11\n",
      "\t\tRound 2: -32.77 ATTTGAGGAACT T:12.0(0.34%),B:3.4(0.01%),P:1e-14\n",
      "\t\tRound 3: -35.81 ATCTAAGGAACT T:11.0(0.31%),B:0.8(0.00%),P:1e-15\n",
      "\t\tRound 4: -35.81 ATCTAAGGAACT T:11.0(0.31%),B:0.8(0.00%),P:1e-15\n",
      "\t\t=Final=: -40.46 ATCTAAGGAACT T:11.0(0.31%),B:0.8(0.00%),P:1e-17\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t19 of 25 Initial Sequence: GAGGAAGTAATG... (-23.049)\n",
      "\t\tRound 1: -23.05 GAGGAAGTAATG T:8.0(0.23%),B:2.9(0.01%),P:1e-10\n",
      "\t\tRound 2: -23.05 GAGGAAGTAATG T:8.0(0.23%),B:2.9(0.01%),P:1e-10\n",
      "\t\t=Final=: -12.42 GAGGAAGTAATG T:8.0(0.23%),B:7.3(0.03%),P:1e-5\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t20 of 25 Initial Sequence: TGGGCGGGGCCC... (-19.041)\n",
      "\t\tRound 1: -4.46 TGGGCGGGGCCC T:103.0(2.89%),B:633.2(2.28%),P:1e-1\n",
      "\t\tRound 2: -6.79 TGGGCGGGCCCC T:58.0(1.64%),B:290.4(1.05%),P:1e-2\n",
      "\t\tRound 3: -13.29 TGGGCGGGCCYS T:74.0(2.09%),B:316.2(1.14%),P:1e-5\n",
      "\t\tRound 4: -17.63 TGGGCGGGCCCC T:79.0(2.23%),B:308.2(1.11%),P:1e-7\n",
      "\t\tRound 5: -18.14 TGGGCGGGCCCC T:70.0(1.97%),B:256.3(0.93%),P:1e-7\n",
      "\t\tRound 6: -18.14 TGGGCGGGCCCC T:70.0(1.97%),B:256.3(0.93%),P:1e-7\n",
      "\t\t=Final=: -18.75 TGGGCGGGCCCC T:96.0(2.74%),B:404.7(1.45%),P:1e-8\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t21 of 25 Initial Sequence: ATTACACTTCAG... (-19.041)\n",
      "\t\tRound 1: -28.54 ATTACACTTCAG T:13.0(0.37%),B:5.7(0.02%),P:1e-12\n",
      "\t\tRound 2: -28.54 ATTACACTTCAG T:13.0(0.37%),B:5.7(0.02%),P:1e-12\n",
      "\t\t=Final=: -29.21 ATTACACTTCAG T:13.0(0.37%),B:5.7(0.02%),P:1e-12\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t22 of 25 Initial Sequence: GAGAATAACCCC... (-19.041)\n",
      "\t\tRound 1: -23.05 VAGAATAACCCH T:8.0(0.23%),B:0.9(0.00%),P:1e-10\n",
      "\t\tRound 2: -35.81 GAGAGTAACCCC T:11.0(0.31%),B:2.8(0.01%),P:1e-15\n",
      "\t\tRound 3: -35.81 GAGAGTAACCCC T:11.0(0.31%),B:2.8(0.01%),P:1e-15\n",
      "\t\t=Final=: -32.95 GAGAGTAACCCC T:11.0(0.31%),B:2.8(0.01%),P:1e-14\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t23 of 25 Initial Sequence: ACTTCCTCCTCC... (-19.041)\n",
      "\t\tRound 1: -10.04 ACTTCCTCCTCC T:65.0(1.84%),B:295.5(1.07%),P:1e-4\n",
      "\t\tRound 2: -10.23 ACTTCCTCCTCC T:83.0(2.34%),B:403.1(1.46%),P:1e-4\n",
      "\t\tRound 3: -10.96 ACTTCCTCCTCC T:90.0(2.53%),B:432.8(1.56%),P:1e-4\n",
      "\t\tRound 4: -11.80 AMTGCCTCCTCC T:51.0(1.44%),B:198.1(0.72%),P:1e-5\n",
      "\t\tRound 5: -14.14 ACHKCCTCYTCC T:65.0(1.84%),B:258.5(0.93%),P:1e-6\n",
      "\t\tRound 6: -14.14 ACHKCCTCYTCC T:65.0(1.84%),B:258.5(0.93%),P:1e-6\n",
      "\t\t=Final=: -14.57 ACHKCCTCYTCC T:64.0(1.82%),B:257.8(0.92%),P:1e-6\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t24 of 25 Initial Sequence: TACTTGAGGGCA... (-18.627)\n",
      "\t\tRound 1: -31.45 TACTTGAGGGCA T:10.0(0.28%),B:1.8(0.00%),P:1e-13\n",
      "\t\tRound 2: -32.77 TACTTGAGGTCA T:12.0(0.34%),B:3.8(0.01%),P:1e-14\n",
      "\t\tRound 3: -32.77 TACTTGAGGTCA T:12.0(0.34%),B:3.8(0.01%),P:1e-14\n",
      "\t\t=Final=: -32.07 TACTTGAGGTCA T:12.0(0.34%),B:3.8(0.01%),P:1e-13\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t25 of 25 Initial Sequence: TACATCCCATTT... (-18.308)\n",
      "\t\tRound 1: -19.04 TACATCCCATTT T:7.0(0.20%),B:1.3(0.00%),P:1e-8\n",
      "\t\tRound 2: -19.04 TACATCCCATTT T:7.0(0.20%),B:1.3(0.00%),P:1e-8\n",
      "\t\t=Final=: -23.17 TACATCCCATTT T:7.0(0.20%),B:1.3(0.00%),P:1e-10\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\n",
      "\tFinalizing Enrichment Statistics (new in v3.4)\n",
      "\tReading input files...\n",
      "\t32625 total sequences read\n",
      "\tCache length = 11180\n",
      "\tUsing binomial scoring\n",
      "\tChecking enrichment of 25 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tOutput in file: getDiff_Nathan_BMDM_K27ac_GW_gained_20_FC1_5_p0001_GC_matched.txt/homerMotifs.motifs12\n",
      "\n",
      "\n",
      "\tNumber of Trial motifs (-T) set to 24 (from 10) to work well with 24 CPUs\n",
      "\t-blen automatically set to 2\n",
      "\tScanning input files...\n",
      "\tParsing sequences...\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\tTotal number of Oligos: 4774183\n",
      "\tAutoadjustment for sequence coverage in background: 0.98x\n",
      "\n",
      "\tOligos: 4774183 of 6133500 max\n",
      "\tTree  : 13293196 of 30667500 max\n",
      "\tOptimizing memory usage...\n",
      "\tCache length = 11180\n",
      "\tUsing binomial scoring\n",
      "\n",
      "\tGlobal Optimization Phase: Looking for enriched oligos with up to 3 mismatches...\n",
      "\n",
      "\tScreening oligos 4774183 (allowing 0 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t87.47% skipped, 12.53% checked (598274 of 4774183), of those checked:\n",
      "\t\t87.47% not in target, 0.00% increased p-value, 0.00% high p-value\n",
      "\n",
      "\tScreening oligos 4774183 (allowing 1 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t87.47% skipped, 12.53% checked (598274 of 4774183), of those checked:\n",
      "\t\t0.00% not in target, 10.01% increased p-value, 0.08% high p-value\n",
      "\n",
      "\tScreening oligos 4774183 (allowing 2 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t97.49% skipped, 2.51% checked (119665 of 4774183), of those checked:\n",
      "\t\t0.00% not in target, 1.88% increased p-value, 0.00% high p-value\n",
      "\n",
      "\tScreening oligos 4774183 (allowing 3 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t99.37% skipped, 0.63% checked (30107 of 4774183), of those checked:\n",
      "\t\t0.00% not in target, 0.27% increased p-value, 0.00% high p-value\n",
      "\tReading input files...\n",
      "\t32625 total sequences read\n",
      "\tCache length = 11180\n",
      "\tUsing binomial scoring\n",
      "\n",
      "\tLocal Optimization Phase:\n",
      "\t1 of 25 Initial Sequence: AGGCCCCGCCCCTT... (-144.539)\n",
      "\t\tRound 1: -273.05 AGGCCCCGCCCCTT T:1215.0(29.26%),B:4164.2(14.06%),P:1e-118\n",
      "\t\tRound 2: -341.50 AGGCCCCGCCCCYY T:1626.0(37.08%),B:5584.1(18.39%),P:1e-148\n",
      "\t\tRound 3: -378.41 AGGCCCCGCCCCCN T:1959.0(42.78%),B:6837.6(22.03%),P:1e-164\n",
      "\t\tRound 4: -395.40 AGGCCCCGCCCCCY T:2044.0(44.15%),B:7088.0(22.74%),P:1e-171\n",
      "\t\tRound 5: -419.43 HGGCCCCGCCCCCB T:2242.0(47.21%),B:7793.0(24.70%),P:1e-182\n",
      "\t\tRound 6: -419.43 HGGCCCCGCCCCCB T:2242.0(47.21%),B:7793.0(24.70%),P:1e-182\n",
      "\t\t=Final=: -345.76 HGGCCCCGCCCCCB T:1328.0(37.83%),B:5282.4(18.89%),P:1e-150\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t2 of 25 Initial Sequence: CCACTTCCTCTTTT... (-119.713)\n",
      "\t\tRound 1: -249.84 CCACTTCCTCTTTT T:1249.0(29.95%),B:4504.9(15.12%),P:1e-108\n",
      "\t\tRound 2: -296.05 YCACTTCCTCTTTY T:734.0(18.87%),B:1885.8(6.63%),P:1e-128\n",
      "\t\tRound 3: -296.05 YCACTTCCTCTTTY T:734.0(18.87%),B:1885.8(6.63%),P:1e-128\n",
      "\t\t=Final=: -283.02 YCACTTCCTCTTTY T:644.0(18.35%),B:1820.2(6.51%),P:1e-122\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t3 of 25 Initial Sequence: ACTGCGCATGCGCA... (-107.849)\n",
      "\t\tRound 1: -145.38 ACTGCGCATGCGCA T:725.0(18.66%),B:2708.6(9.39%),P:1e-63\n",
      "\t\tRound 2: -145.38 ACTGCGCATGCGCA T:725.0(18.66%),B:2708.6(9.39%),P:1e-63\n",
      "\t\t=Final=: -73.33 ACTGCGCATGCGCA T:263.0(7.49%),B:932.4(3.33%),P:1e-31\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t4 of 25 Initial Sequence: ACTACAACTCCCAG... (-96.788)\n",
      "\t\tRound 1: -130.36 ACTACAACTCCCAG T:189.0(5.24%),B:345.1(1.25%),P:1e-56\n",
      "\t\tRound 2: -130.36 ACTACAACTCCCAG T:189.0(5.24%),B:345.1(1.25%),P:1e-56\n",
      "\t\t=Final=: -102.53 ACTACAACTCCCAG T:138.0(3.93%),B:250.2(0.89%),P:1e-44\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t5 of 25 Initial Sequence: CCTCTGATTGGTCG... (-46.601)\n",
      "\t\tRound 1: -99.21 CCTCTGATTGGTCG T:727.0(18.71%),B:3136.7(10.79%),P:1e-43\n",
      "\t\tRound 2: -171.76 CCTCTGATTGGTCG T:831.0(21.08%),B:3041.5(10.48%),P:1e-74\n",
      "\t\tRound 3: -171.76 CCTCTGATTGGTCG T:831.0(21.08%),B:3041.5(10.48%),P:1e-74\n",
      "\t\t=Final=: -131.06 CCTCTGATTGGTCG T:371.0(10.57%),B:1168.8(4.18%),P:1e-56\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t6 of 25 Initial Sequence: GTGACGTCACCACG... (-44.265)\n",
      "\t\tRound 1: -76.93 GTGACGTCACCACG T:268.0(7.35%),B:884.6(3.17%),P:1e-33\n",
      "\t\tRound 2: -76.93 GTGACGTCACCACG T:268.0(7.35%),B:884.6(3.17%),P:1e-33\n",
      "\t\t=Final=: -59.57 GTGACGTCACCACG T:163.0(4.64%),B:501.8(1.79%),P:1e-25\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t7 of 25 Initial Sequence: CAATTCTCGCGAGA... (-39.899)\n",
      "\t\tRound 1: -58.09 CAAYTCTCGCGAGA T:99.0(2.78%),B:211.9(0.77%),P:1e-25\n",
      "\t\tRound 2: -58.09 CAAYTCTCGCGAGA T:99.0(2.78%),B:211.9(0.77%),P:1e-25\n",
      "\t\t=Final=: -32.17 CAAYTCTCGCGAGA T:55.0(1.57%),B:126.4(0.45%),P:1e-13\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t8 of 25 Initial Sequence: GCCACCAGGGGGCG... (-37.436)\n",
      "\t\tRound 1: -50.08 GCCACCAGGGGGCG T:80.0(2.25%),B:165.6(0.60%),P:1e-21\n",
      "\t\tRound 2: -59.99 GCCTCCAGGGGGCG T:168.0(4.67%),B:500.3(1.80%),P:1e-26\n",
      "\t\tRound 3: -59.99 GCCTCCAGGGGGCG T:168.0(4.67%),B:500.3(1.80%),P:1e-26\n",
      "\t\t=Final=: -57.03 GCCTCCAGGGGGCG T:160.0(4.56%),B:499.2(1.79%),P:1e-24\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t9 of 25 Initial Sequence: GGGCGGGGCGGGGC... (-31.445)\n",
      "\t\tRound 1: -2.80 GGGCGGGGCGGGGC T:47.0(1.33%),B:284.4(1.03%),P:1e-1\n",
      "\t\tRound 2: -3.92 GGGCSGGGCGGGGC T:8.0(0.23%),B:22.5(0.08%),P:1e-1\n",
      "\t\tRound 3: -9.28 GGGCCGGGCCGGGS T:28.0(0.79%),B:93.0(0.34%),P:1e-4\n",
      "\t\tRound 4: -9.28 GGGCCGGGCCGGGS T:28.0(0.79%),B:93.0(0.34%),P:1e-4\n",
      "\t\t=Final=: -9.00 GGGCCGGGCCGGGS T:16.0(0.46%),B:43.0(0.15%),P:1e-3\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t10 of 25 Initial Sequence: CGGCCGCCATTTTG... (-29.207)\n",
      "\t\tRound 1: -39.15 CGGCCGCCATYTTG T:155.0(4.32%),B:552.2(1.99%),P:1e-17\n",
      "\t\tRound 2: -44.66 CSGCCGCCATCTTG T:133.0(3.72%),B:412.9(1.49%),P:1e-19\n",
      "\t\tRound 3: -52.88 VSGCCGCCATCTTG T:262.0(7.19%),B:1014.5(3.62%),P:1e-22\n",
      "\t\tRound 4: -53.32 MVGCCGCCATCTTS T:189.0(5.24%),B:637.2(2.29%),P:1e-23\n",
      "\t\tRound 5: -57.60 CGGCCGCCATCTTG T:201.0(5.57%),B:669.8(2.41%),P:1e-25\n",
      "\t\tRound 6: -57.60 CGGCCGCCATCTTG T:201.0(5.57%),B:669.8(2.41%),P:1e-25\n",
      "\t\t=Final=: -54.44 CGGCCGCCATCTTG T:196.0(5.58%),B:696.4(2.49%),P:1e-23\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t11 of 25 Initial Sequence: TGACCAGTAGTAAC... (-27.824)\n",
      "\t\tRound 1: -60.75 TGACCMGTAGTRAC T:45.0(1.27%),B:39.2(0.14%),P:1e-26\n",
      "\t\tRound 2: -83.66 TGACCTRGAGTRAC T:21.0(0.60%),B:2.0(0.01%),P:1e-36\n",
      "\t\tRound 3: -83.66 TGACCTRGAGTRAC T:21.0(0.60%),B:2.0(0.01%),P:1e-36\n",
      "\t\t=Final=: -74.70 TGACCTRGAGTRAC T:21.0(0.60%),B:2.0(0.01%),P:1e-32\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t12 of 25 Initial Sequence: GCACAACTGAAAAT... (-27.187)\n",
      "\t\tRound 1: -31.45 GCACAACTGAAAAT T:10.0(0.28%),B:0.0(0.00%),P:1e-13\n",
      "\t\tRound 2: -31.45 GCACAACTGAAAAT T:10.0(0.28%),B:0.0(0.00%),P:1e-13\n",
      "\t\t=Final=: -35.98 GCACAACTGAAAAT T:10.0(0.28%),B:0.0(0.00%),P:1e-15\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t13 of 25 Initial Sequence: AAGATGGCGGCGCC... (-23.046)\n",
      "\t\tRound 1: -4.55 ARGATGGCGGCGCC T:9.0(0.26%),B:24.7(0.09%),P:1e-1\n",
      "\t\tRound 2: -5.52 AGGATGGCGGCGBC T:6.0(0.17%),B:9.9(0.03%),P:1e-2\n",
      "\t\tRound 3: -5.52 AGGATGGCGGCGBC T:6.0(0.17%),B:9.9(0.03%),P:1e-2\n",
      "\t\t=Final=: -6.81 AGGATGGCGGCGBC T:6.0(0.17%),B:9.9(0.04%),P:1e-2\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t14 of 25 Initial Sequence: GGTGGGCGGGGCCC... (-19.038)\n",
      "\t\tRound 1: -2.55 NNHNGDHDDKNNDD T:31.0(0.88%),B:178.4(0.65%),P:1e-1\n",
      "\t\tRound 2: -2.55 NNHNGDHDDKNNDD T:31.0(0.88%),B:178.4(0.65%),P:1e-1\n",
      "\t\t=Final=: -4.25 NNHNGDHDDKNNDD T:16.0(0.46%),B:68.4(0.24%),P:1e-1\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t15 of 25 Initial Sequence: CGAGTACCAATTGG... (-18.624)\n",
      "\t\tRound 1: -31.45 CGAGTACCAATTGG T:10.0(0.28%),B:2.7(0.01%),P:1e-13\n",
      "\t\tRound 2: -31.45 CGAGTACCAATTGG T:10.0(0.28%),B:2.7(0.01%),P:1e-13\n",
      "\t\t=Final=: -31.60 CGAGTACCAATTGG T:9.0(0.26%),B:1.5(0.01%),P:1e-13\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t16 of 25 Initial Sequence: TCAGCCAATAGGAA... (-16.852)\n",
      "\t\tRound 1: -35.81 TCAGCCAATAGGAA T:11.0(0.31%),B:2.8(0.01%),P:1e-15\n",
      "\t\tRound 2: -35.81 TCAGCCAATAGGAA T:11.0(0.31%),B:2.8(0.01%),P:1e-15\n",
      "\t\t=Final=: -28.24 TCAGCCAATAGGAA T:16.0(0.46%),B:10.2(0.04%),P:1e-12\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t17 of 25 Initial Sequence: ATGAGGAACTTCCA... (-16.529)\n",
      "\t\tRound 1: -44.11 ATGAGGAACTTCCA T:56.0(1.58%),B:94.1(0.34%),P:1e-19\n",
      "\t\tRound 2: -44.11 ATGAGGAACTTCCA T:56.0(1.58%),B:94.1(0.34%),P:1e-19\n",
      "\t\t=Final=: -43.49 ATGAGGAACTTCCA T:55.0(1.57%),B:96.1(0.34%),P:1e-18\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t18 of 25 Initial Sequence: GACGTGATTGACAG... (-16.346)\n",
      "\t\tRound 1: -35.68 GASGTGATTGACAG T:17.0(0.48%),B:7.7(0.03%),P:1e-15\n",
      "\t\tRound 2: -35.68 GASGTGATTGACAG T:17.0(0.48%),B:7.7(0.03%),P:1e-15\n",
      "\t\t=Final=: -36.57 GASGTGATTGACAG T:17.0(0.48%),B:7.7(0.03%),P:1e-15\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t19 of 25 Initial Sequence: CCAGAAGAGGGCGC... (-15.890)\n",
      "\t\tRound 1: -23.05 CCAGAAGAGGGCGC T:8.0(0.23%),B:2.6(0.01%),P:1e-10\n",
      "\t\tRound 2: -23.05 CCAGAAGAGGGCGC T:8.0(0.23%),B:2.6(0.01%),P:1e-10\n",
      "\t\t=Final=: -23.17 CCAGAAGAGGGCGC T:7.0(0.20%),B:2.0(0.01%),P:1e-10\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t20 of 25 Initial Sequence: GCAGAACTAACAGG... (-15.719)\n",
      "\t\tRound 1: -31.77 GCAGAACTAACAGG T:14.0(0.40%),B:5.1(0.02%),P:1e-13\n",
      "\t\tRound 2: -31.77 GCAGAACTAACAGG T:14.0(0.40%),B:5.1(0.02%),P:1e-13\n",
      "\t\t=Final=: -29.21 GCAGAACTAACAGG T:13.0(0.37%),B:5.1(0.02%),P:1e-12\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t21 of 25 Initial Sequence: CTTCCGCTTCCGGC... (-15.685)\n",
      "\t\tRound 1: -20.83 CTTCCGCTTCCGGC T:46.0(1.30%),B:123.7(0.45%),P:1e-9\n",
      "\t\tRound 2: -20.83 CTTCCGCTTCCGGC T:46.0(1.30%),B:123.7(0.45%),P:1e-9\n",
      "\t\t=Final=: -21.00 CTTCCGCTTCCGGC T:46.0(1.31%),B:128.3(0.46%),P:1e-9\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t22 of 25 Initial Sequence: CCTTAAAGGGGCCG... (-15.617)\n",
      "\t\tRound 1: -26.86 CCTTAAAGGGGCCG T:32.0(0.91%),B:51.4(0.19%),P:1e-11\n",
      "\t\tRound 2: -26.86 CCTTAAAGGGGCCG T:32.0(0.91%),B:51.4(0.19%),P:1e-11\n",
      "\t\t=Final=: -27.18 CCTTAAAGGGGCCG T:30.0(0.85%),B:47.1(0.17%),P:1e-11\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t23 of 25 Initial Sequence: CTGTGGGAGGGACG... (-15.578)\n",
      "\t\tRound 1: -29.08 CTGTGGGAGGGACG T:18.0(0.51%),B:12.7(0.04%),P:1e-12\n",
      "\t\tRound 2: -29.08 CTGTGGGAGGGACG T:18.0(0.51%),B:12.7(0.04%),P:1e-12\n",
      "\t\t=Final=: -30.48 CTGTGGGAGGGACG T:18.0(0.51%),B:12.7(0.05%),P:1e-13\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t24 of 25 Initial Sequence: GGGGGCGGAGCCTC... (-15.184)\n",
      "\t\tRound 1: -7.30 NSGGGCGGAGCCNN T:11.0(0.31%),B:24.0(0.08%),P:1e-3\n",
      "\t\tRound 2: -12.23 GCGGGCGGAGCCWS T:11.0(0.31%),B:13.3(0.05%),P:1e-5\n",
      "\t\tRound 3: -14.76 GCGGGCGGAGCCWG T:11.0(0.31%),B:10.5(0.04%),P:1e-6\n",
      "\t\tRound 4: -14.76 GCGGGCGGAGCCWG T:11.0(0.31%),B:10.5(0.04%),P:1e-6\n",
      "\t\t=Final=: -15.22 GCGGGCGGAGCCWG T:11.0(0.31%),B:11.2(0.04%),P:1e-6\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t25 of 25 Initial Sequence: AGGGGCGGGGCCGG... (-15.184)\n",
      "\t\tRound 1: -12.49 AGGGGCGGGGCCGG T:38.0(1.08%),B:125.3(0.45%),P:1e-5\n",
      "\t\tRound 2: -12.49 AGGGGCGGGGCCGG T:38.0(1.08%),B:125.3(0.45%),P:1e-5\n",
      "\t\t=Final=: -12.88 AGGGGCGGGGCCGG T:37.0(1.05%),B:124.1(0.44%),P:1e-5\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\n",
      "\tFinalizing Enrichment Statistics (new in v3.4)\n",
      "\tReading input files...\n",
      "\t32625 total sequences read\n",
      "\tCache length = 11180\n",
      "\tUsing binomial scoring\n",
      "\tChecking enrichment of 25 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tOutput in file: getDiff_Nathan_BMDM_K27ac_GW_gained_20_FC1_5_p0001_GC_matched.txt/homerMotifs.motifs14\n",
      "\n",
      "\n",
      "\tNumber of Trial motifs (-T) set to 24 (from 10) to work well with 24 CPUs\n",
      "\t-blen automatically set to 3\n",
      "\tScanning input files...\n",
      "\tParsing sequences...\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\tTotal number of Oligos: 5254150\n",
      "\tAutoadjustment for sequence coverage in background: 0.98x\n",
      "\n",
      "\tOligos: 5254150 of 6068250 max\n",
      "\tTree  : 15848856 of 30341250 max\n",
      "\tOptimizing memory usage...\n",
      "\tCache length = 11180\n",
      "\tUsing binomial scoring\n",
      "\n",
      "\tGlobal Optimization Phase: Looking for enriched oligos with up to 3 mismatches...\n",
      "\n",
      "\tScreening oligos 5254150 (allowing 0 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t88.53% skipped, 11.47% checked (602756 of 5254150), of those checked:\n",
      "\t\t88.53% not in target, 0.00% increased p-value, 0.00% high p-value\n",
      "\n",
      "\tScreening oligos 5254150 (allowing 1 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t88.53% skipped, 11.47% checked (602756 of 5254150), of those checked:\n",
      "\t\t0.00% not in target, 5.55% increased p-value, 0.01% high p-value\n",
      "\n",
      "\tScreening oligos 5254150 (allowing 2 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t94.08% skipped, 5.92% checked (311202 of 5254150), of those checked:\n",
      "\t\t0.00% not in target, 4.75% increased p-value, 0.00% high p-value\n",
      "\n",
      "\tScreening oligos 5254150 (allowing 3 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t98.83% skipped, 1.17% checked (61707 of 5254150), of those checked:\n",
      "\t\t0.00% not in target, 0.90% increased p-value, 0.00% high p-value\n",
      "\tReading input files...\n",
      "\t32625 total sequences read\n",
      "\tCache length = 11180\n",
      "\tUsing binomial scoring\n",
      "\n",
      "\tLocal Optimization Phase:\n",
      "\t1 of 25 Initial Sequence: ACTACAACTCCCAGAA... (-94.618)\n",
      "\t\tRound 1: -138.99 ACTACAACTCCCAGAA T:164.0(4.57%),B:245.6(0.89%),P:1e-60\n",
      "\t\tRound 2: -138.99 ACTACAACTCCCAGAA T:164.0(4.57%),B:245.6(0.89%),P:1e-60\n",
      "\t\t=Final=: -110.82 ACTACAACTCCCAGAA T:136.0(3.87%),B:225.4(0.81%),P:1e-48\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t2 of 25 Initial Sequence: CTCAGGCCCCGCCCAC... (-85.395)\n",
      "\t\tRound 1: -220.56 CTCAGGCCCCGCCCAC T:1891.0(41.66%),B:8084.3(25.50%),P:1e-95\n",
      "\t\tRound 2: -278.83 CBCHGGCCCCGCCCCC T:2083.0(44.76%),B:8394.8(26.34%),P:1e-121\n",
      "\t\tRound 3: -306.96 CYCAGGCCCCGCCCCC T:1792.0(39.99%),B:6663.0(21.54%),P:1e-133\n",
      "\t\tRound 4: -317.29 CCCAGGCCCCGCCCCC T:2182.0(46.30%),B:8472.6(26.55%),P:1e-137\n",
      "\t\tRound 5: -334.94 CSCHGGCCCCGCCCCC T:2001.0(43.46%),B:7415.8(23.66%),P:1e-145\n",
      "\t\tRound 6: -352.81 CBCHGGCCCCGCCCCC T:1976.0(43.05%),B:7135.3(22.88%),P:1e-153\n",
      "\t\tRound 7: -352.81 CBCHGGCCCCGCCCCC T:1976.0(43.05%),B:7135.3(22.88%),P:1e-153\n",
      "\t\t=Final=: -291.44 CBCHGGCCCCGCCCCC T:1262.0(35.95%),B:5236.6(18.73%),P:1e-126\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t3 of 25 Initial Sequence: AGAAAGAGGAAGTGAA... (-64.888)\n",
      "\t\tRound 1: -161.12 AGAAAGAGGAAGTGAA T:730.0(18.78%),B:2612.8(9.07%),P:1e-69\n",
      "\t\tRound 2: -225.79 RGAAAGAGGAAGTGAV T:692.0(17.90%),B:2035.1(7.14%),P:1e-98\n",
      "\t\tRound 3: -229.81 RRRARGAGGAAGTGRN T:1082.0(26.53%),B:3840.2(13.05%),P:1e-99\n",
      "\t\tRound 4: -229.81 RRRARGAGGAAGTGRN T:1082.0(26.53%),B:3840.2(13.05%),P:1e-99\n",
      "\t\t=Final=: -206.35 RRRARGAGGAAGTGRN T:868.0(24.73%),B:3437.3(12.29%),P:1e-89\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t4 of 25 Initial Sequence: CAGTGCGCGTGCGCAG... (-33.818)\n",
      "\t\tRound 1: -67.95 CAGTGCGCGTGCGCAG T:619.0(16.17%),B:2877.9(9.95%),P:1e-29\n"
     ]
    }
   ],
   "source": [
    "!findMotifsGenome.pl getDiff_Nathan_BMDM_K27ac_GW_gained_20_FC1_5_p0001.txt mm10r getDiff_Nathan_BMDM_K27ac_GW_gained_20_FC1_5_p0001_GC_matched.txt -size 200 -len 8,10,12,14,16 -mis 3 -p 24"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.6"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
